{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Parsing of the dataset CORD-19\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Importing the parser and parsing the datasets"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Parser import *"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Creating a Parser and specifying what kind of dataset we want to parse\n",
    "parser = Parser([Dataset.BIORXIV])\n",
    "parser.parse(indexByFile = False);"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Example of accesing the paper by index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[1mTitle\u001b[0m\n",
      "How domain growth is implemented determines the long term behaviour of a cell population through its effect on spatial correlations\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Domain growth plays an important role in many biological systems, and so the inclusion of domain growth in models of these biological systems is important to understanding how these biological systems function. In this work we present methods to include the effects of domain growth on the evolution of spatial correlations in a continuum approximation of a lattice-based model of cell motility and proliferation. We show that, depending on the way in which domain growth is implemented, different steady-state densities are predicted for an agent population. Furthermore, we demonstrate that the way in which domain growth is implemented can result in the evolution of the agent density depending on the size of the domain. Continuum approximations that ignore spatial correlations cannot capture these behaviours, while those that account for spatial correlations do. These results will be of interest to researchers in developmental biology, as they suggest that the nature of domain growth can determine the characteristics of cell populations. * ross@maths.ox.ac.ukIn this work we examine how domain growth affects the evolution of spatial correlations between agents in IBMs, and importantly, how the way in which domain growth is implemented affects spatial correlations differently. To do so we use an agent-based (our agents represent 2 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint N x (t) by N y (t), where N x (t) is the number of lattice sites in a row and N y (t) is the number of sites in a column. All simulations are performed with periodic boundary conditions, and throughout this work the lattice spacing, ∆, is equal to one. For notational convenience we shall now write N x (t) as N x , and N y (t) as N y .In the IBM each agent is initially assigned to a lattice site, from which it can move or proliferate into an adjacent site. If an agent attempts to move into a site that is already occupied, 3 author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\u001b[1mBody\u001b[0m\n",
      "Growth is of fundamental importance in biological systems [1, 2] . From embryonic development to tissue renewal, growth plays a central role in the development, maintenance and repair of biological systems throughout their lifetime [3] . Types of isotropic growth that have been observed in biological systems include exponential, linear and logistic [4] [5] [6] [7] [8] [9] [10] [11] . Anisotropic growth also plays an important role in biological systems. For instance apical growth, whereby the tip region of a domain grows, is observed in plant root extension and chick limb outgrowth [12, 13] .Importantly, growth has also been shown to contribute in the development of biological systems in different ways. Experiments have shown that the advective component of domain growth is important in enabling migrating cells to reach their target site [6] [7] [8] [9] . Meanwhile, theoretical studies have suggested that differential growth rates in developing tissue allow the generation of diverse biological forms and structures [10, 14] . Consequently, it is important to include domain growth in many models of biological systems [1, 2, [6] [7] [8] [9] [14] [15] [16] [17] [18] .Spatial correlations are often observed in biological systems [19] [20] [21] [22] [23] [24] . For instance, in cell populations spatial correlations can be established by cell proliferation, as a new cell is naturally close to its parent cell following division. Individual-based models (IBMs) are able to recapitulate these spatial correlations, whereas models that employ certain mean-field approximations (MFAs), such as the logistic model [25] , and, in a spatially-extended context, the diffusion equation, cannot [26] [27] [28] [29] [30] . Accurate continuum models are important tools for understanding the behaviour of biological systems as, in contrast to IBMs, they generally allow for greater mathematical analysis. This analysis can be crucial in forming a mechanistic understanding of biological systems, which is not always apparent from simply studying the results of a large number of repeats of an IBM. Therefore, in order to derive accurate continuum approximations of IBMs that include cell proliferation it is often necessary to account for the effects of spatial correlations [25, [31] [32] [33] [34] [35] [36] [37] [38] [39] . cells), discrete random-walk model on a two-dimensional square lattice with volume-exclusion.We had reasoned that a standard MFA would not be able to capture the behaviours exhibited by the IBM, and that this standard MFA would require correction by the inclusion of spatial correlations in the form of a correlation ordinary differential equation (ODE) to accurately approximate the IBM results. This last point has been shown previously in scenarios without growth [25] .The outline of this work is as follows: to begin with we introduce the IBM and growth mechanisms in Section 2.1. We then define our individual and pairwise density functions, and derive a system of ODEs describing the evolution of the individual and pairwise density functions with respect to time on a growing domain in Sections 2.2-2.3. To test the accuracy of these ODE models we compare them with ensemble averages for the evolution of the macroscopic density from the IBM in Section 3. We demonstrate that the standard MFA is not able to accurately capture the effects of domain growth in the IBM, whereas the ODE models that include the effect of spatial correlations do.In this section we introduce the IBM and the domain growth mechanisms we employ throughout this work. We then derive equations describing the evolution of the individual and pairwise density functions in the IBM for both growth mechanisms.The IBM is simulated on a two-dimensional square lattice with lattice spacing ∆ [40] and size to the same overall growth rate when implemented with the same growth rate parameters.(a) (b) Figure 1 : (Colour online). Before and after the growth events for both (a) GM1 and (b) GM2, in which growth is along the x-axis (horizontal direction) for a two-dimensional lattice. In each row the yellow (light grey) site has been chosen to undergo a growth event. Following this the yellow (light grey) site is moved to the right with its contents, for instance an agent (represented by a black cell). The blue (dark grey) sites are the new lattice sites and are always initially empty. The contents of all the other sites remain unaffected, although in some cases their neighbouring sites will change.We define the individual density functions, ρ For instance, (2, 3) would be the lattice site situated in the second row and the third column of the lattice.The following derivation for the individual density functions is the same for GM1 and GM2.The sum of the individual density functions on a domain of size N x × N y at [t + δt) can be 5 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint written in terms of the individual density functions at time t:The terms on the RHS of Eq. (1) correspond to the following events: i) no growth event occurs in [t, t + δt); ii) a growth event occurs in the horizontal (x) direction in [t, t + δt); and iii) a growth event occurs in the vertical (y) direction in [t, t + δt). Due to our initial conditions in the IBM we can assume translational invariance for the probability of an agent occupying a site in this derivation, by this we meanwhere n indexes any other site on the domain. 1 Therefore we can rewrite Eq. (1) asWe can simplify Eq. (3) in the following mannerIf we now rearrange Eq. (4) and take the limit as δt → 0 we obtain dρ Nx×Ny 1If we make the approximation ρ This approximation has been previously published [17] , and reasonably implies that domain growth 'dilutes' the agent density. Finally, we simplify Eq. (6) to obtain dρ Nx×Ny 1 (m; t) dt = −(P gx + P gy )ρ Eq. (7) describes how exponential domain growth affects agent density. It is important to note that Eq. (7) describes how exponential domain growth affects the evolution of individual density functions because we have defined P gx and P gy as constants. In the course of the following derivation it will be useful to write the pairwise density functions in terms of the distances between sites, therefore we shall rewrite the individual density functions asIf we substitute Eq. (8) into Eq. (7) we obtain dc(t) dt = −(P gx + P gy )c(t).Eq. (9) is a MFA describing the effect of domain growth on the evolution agent density [32] . 7 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintWe now derive the terms necessary to include the effect of two-dimensional exponential domain growth in the evolution of the pairwise density functions. Importantly, GM1 and GM2 affect the evolution of the pairwise density functions differently. Figure 2 displays two configurations of two agents, which we will term colinear and diagonal. . In (a), two colinear lattice sites share the same row but not the same column (or vice versa). In (b), two lattice sites are diagonal, meaning they do not share the same row or column. In both (a) and (b) r x is the distance between two lattice sites in the horizontal direction, and r y is the distance between two lattice sites in the vertical direction. In (b) this means r x = 3 and r y = 1.The horizontal and vertical distances between sites is measured from their centres as displayed in Fig. 2 (b), and r x is the distance between two lattice sites in the horizontal direction, and r y is the distance between two lattice sites in the vertical direction.We now derive equations for the evolution of the pairwise density functions for GM1 domain growth. We define the pairwise density functions, ρ (m, m + (r x , r y ); t),where m is (i, j), with i indexing the row number of a lattice site, j indexing the column number of a lattice site, and (r x , r y ) is a vector that represents a fixed displacement as defined in Fig.   8 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint 2.As the initial conditions in the IBM are, on average, spatially uniform we are able to assume translational invariance for the probability of two sites a given displacement being occupied.This means the pairwise density function can be written as a function of the displacement between two lattice sites, (r x , r y ). Therefore, we will further simplify our notation to obtain ρ Nx×Ny 2 (m, m + (r x , r y ); t) = ρ (r x , r y ; t).This 'abuse' of notation, whereby ρ Nx×Ny 2 is rewritten as a function of the displacement between two lattice sites as opposed to the lattice sites themselves, will prove useful in the following derivation.We begin with the colinear component of the equations for the evolution of the pairwise density functions, that is, the scenario in which the lattice sites in question share the same column or row, as depicted in Fig. 2 (a) . For agents colinear in the horizontal direction, that is, r y = 0, the evolution of the pairwise density functions for GM1 isNy j=1 ρ Nx×Ny 2(r x , 0; t)The terms on the RHS represent: i) no growth event occurs in [t, t + δt); ii) a growth event occurs in the horizontal direction between agents (r x − 1, 0) apart, moving them (r x , 0) apart on a domain of size N x × N y at [t + δt); iii) a growth event occurs in the horizontal direction at a site that is not in between agents (r x , 0) apart, meaning that they remain (r x , 0) apart but now on a domain of size N x × N y at time [t + δt); and iv) a growth event occurs in the vertical direction (as the sites are horizontally colinear in this a GM1 growth event cannot change the distance between them).Similarly, the evolution of the colinear component for GM1 for agents colinear in the verti-Ny j=1 ρ Nx×Ny 2 (0, r y ; t)As in the case of the individual density functions, we can simplify Eq. (12) to obtainIf we apply the approximations 2 ((and 2 This approximation sensibly implies that domain growth 'dilutes' pairwise agent densities.10 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint, Eq. (14) can be rewritten asRearranging Eq. (15) and taking the limit as δt → 0 we finish with dρ Nx×Ny 2The equivalent equation for sites colinear in the vertical direction (see Eq. (13)) is11 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintFor the diagonal component for GM1, that is, neither r x , r y = 0, we have, by similar reasoningIf we follow the same procedure as for the colinear component we obtainFor the derivation of the evolution of the pairwise density functions with GM2 we refer the reader to Appendix A and simply state the results here. The evolution equation for the colinear(r x , 0; t)12 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint An analogous equation exists for the vertically colinear component for GM2. The diagonal component for GM2 is(r x , r y ; t)(r x , r y ; t)(r x , r y + 1; t)If we compare Eqs. (16) and (19)- (21) it is apparent that the length of the domain influences the evolution of the pairwise density functions in the case of GM2, but not in GM1. Therefore, we expect the evolution of the pairwise density functions to be affected by the initial domain size in the case of GM2, but not GM1.We initially present results for immotile, non-proliferative agents on a growing domain, in which domain growth is exponential. We simulate the scenario in which all lattice sites are occupied by an agent in the IBM on a lattice of size 100 by 100 sites. This means the initial conditions for Eqs. (16) and (19)- (21) areand ρ Nx×Ny 213 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint As previously stated, N x and N y describe the number of lattice sites in the horizontal and vertical directions, respectively. However, as results from the IBM are ensemble averages we substitute N x and N y for their continuum analogues L x (t) and L y (t), respectively. For exponential domain growth L x (t) evolves according toThe evolution of L y (t) is equivalent. This substitution of N x (t) and N y (t) with L x (t) and L y (t) avoids jump discontinuities in the numerical solutions of Eqs. (16) and (19)- (21), which are not present in the averaged IBM results. To solve our equations numerically we use MATLAB's ode15s.In Fig. 3 the numerical solutions of Eqs. (16) and (19) we see the evolution of the pairwise density functions for GM2, and in this case in the evolution of all the colinear and diagonal components is the same. 14 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintIt has previously been shown how to include the effect of agent motility and proliferation in Eqs. (9) , (16) , and (19)-(21) and so we simply state the result for the individual density function [25, 38, 39] . The evolution of the individual density function for motile and proliferating agents on a growing domain isAs can be seen from Eq. (25) the inclusion of agent proliferation means that pairwise density functions are now present in the equations for the evolution of the individual density functions, which is not the case without agent proliferation (Eq. (9)).To make our results comparable with the large amount of research already completed in this area we display the results in terms of the correlation function [25, [31] [32] [33] [34] [35] [36] [37] [38] [39] . The correlation function [34, 35] is simply defined asThe correlation function is a useful variable to work with as F ≡ 1 means that the occupancy of two lattice sites a given distance apart is independent. If we substitute Eq. (26) into Eq.(25) we obtainThe standard MFA assumes F (1, 0; t) = 1, that is, the effect of spatial correlations is ignored, and so Eq. (27) becomesFor the IBM in all simulations we have an initial uniform random seeding of density 0.05. The lattice state of site i of the two-dimensional IBM is described by variable σ i (i.e. occupied by an 15 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint agent or unoccupied). This means the normalised average agent density for the two-dimensional domain isHere 1 A is the indicator function for species A (i.e. 1 if species A occupies site i, and 0 if it does not). As previously discussed, the initial condition in the IBM is achieved by populating a certain number of sites uniformly at random until the initial density is achieved, the initial condition for the correlation function is thereforeThat is, at all distances there are initially no spatial correlations between agents. We also rescale time to allow for ease of comparison between simulations with different parameters:The rescaling of time is necessary to allow the easy comparison of simulations with different parameters. As before we solve our equations numerically with MATLAB's ode15s.In Fig. 4 the effects of domain growth via GM1 on the evolution of the macroscopic agent density can be seen. We see that the inclusion of GM1 causes the steady-state density predicted by the standard MFA, Eq. (28), to be incorrect. The steady-state density calculated from an ensemble average from the IBM is lower than the MFA predicts, which the correlation ODE model (Eq. (27)) is able to capture. The rate at which the agent density increases is also reduced in the IBM due to spatial correlations, and the correlations ODE model better approximates this than the MFA. One may expect domain growth to decrease spatial correlations by 'breaking-up' existing agent clusters which, in turn, would allow greater agent movement and cause the MFA to be more accurate. However, the nature of GM1 means that it does not break up agents effectively, only potentially 'freeing-up' one lattice site for two agents in each row/column every growth event (see Fig. 1 ). GM1 also does not cause agents to make 16 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint new neighbouring agents, with agents and their neighbours moving synchronously in a growth event (except at the position of growth where two columns/rows will be separated). Therefore, the spatial correlations associated with agent proliferation (in conjunction with the dilution of agent density due to domain growth) lower the steady-state density of the agents more than the displacement of agents by domain growth counteracts the spatial correlations established by agent proliferation. In Fig. 5 the effect of GM1 on spatial correlations in both the correlation ODE model and the IBM can be seen. Importantly, we see that as the growth rate is increased the spatial correlations augment. From Fig. 4 this result is to be expected, as domain growth lowers the agent steady-state density but is not effective at reducing spatial correlations. A similar result has previously been observed in the case of agent death [25] . In addition, at the agent steady-state density the spatial correlations do not return to F = 1. This explains why the MFA incorrectly predicts the agent steady-state density, and shows why the correlation ODE model is able to correctly approximate the steady-state density of the IBM. This is because the MFA is only correct when F = 1, which is not the case in the IBM in this scenario with GM1.17 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint In Fig. 6 the effect of domain growth via GM2 on agent density can be seen. In this case we see that GM2 causes the steady-state density predicted by the MFA to be correct. This is because GM2 is more effective at breaking up agent clusters than GM1, potentially 'freeing-up' one lattice site for four agents in each row/column every growth event. This counteracts the effect of the spatial correlations created by agent proliferation. GM2 also causes agents to change neighbouring agents in the IBM, and so further reduces the spatial correlations associated with agent proliferation. Despite this we see that the correlations ODE model still more accurately predicts the rate at which the agent density evolves, while also accurately approximating the steady-state density in the IBM. In Fig. 7 the effect of GM2 on spatial correlations can be seen in both the correlations ODE model and calculated directly from the IBM. In contrast to GM1 we see that as the growth rate is increased the spatial correlations decrease. We also see that at the steady-state density F = 1, this provides the reason as to why the agent steady-state predicted by the MFA is correct.19 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintIn Fig. 8 we can see that the size of the domain influences the evolution of the agent density in the case of GM2, but not in the case of GM1. In the case of GM2, as the initial domain size is increased the evolution of the macroscopic agent density is accelerated. This is because colinear spatial correlations, established by agent proliferation, are broken down proportional to the domain size in GM2 (Eq. (20) ).Finally, in Fig. 9 we see the initial domain size also impacts the evolution of the spatial correlations for GM2, but not for GM1 (the figure displaying this is in Appendix B). In the case of GM2 as the initial domain size is increased the spatial correlations between agents are reduced. 21 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintThe importance of accurately including the effect of domain growth in models of biological phenomena is becoming increasingly apparent [1, 2, 6, [14] [15] [16] [17] . Many of the fundamental developmental processes that determine mammalian morphology are either characterised by growth, or occur during it, and so it is possible that growth may play an important role in determining and coordinating cell population behaviour [3, 9] .In this work we have studied two different implementations of domain growth. We chose two different, yet potentially biologically relevant growth mechanisms [42] [43] [44] , to highlight how understanding the form of domain growth is crucial. These growth mechanisms originated from thinking about the different ways in which new sites could be added to the IBM domain, and the consequences this may have on agent density. As has been shown in this work, GM1 is an implementation of domain growth that conserves colinear correlations in comparison to diagonal correlations, and GM2 is one that reduces colinear correlations as a rate proportional to the size of the domain. This difference between GM1 and GM2 is why GM2 exhibits a length dependency and GM1 does not. It is important to stress we have used GM1 as an extreme example of growth in which the cell cycles of adjacent cells that are causing the domain to grow are synchronous (i.e. the cells in the underlying tissue), and GM2 as an example of when they are not. In reality, it is unlikely that domain growth in biological systems can be captured by algorithms as simple as GM1 and GM2. However, they represent the simplest and most tractable domain growth mechanisms for this initial study.It is important to acknowledge that we have only looked at scenarios where P p = P m . This case represents a very high rate of cell proliferation relative to motility in a biological context. However, we chose these parameters to highlight the accuracy of our approximate correlation ODE model. In situations where P p < P m the correlation ODE model approximates the IBM results even better, as has been shown previously [25, 39] . We have also only looked at the case when P gx = P gy in order to reduce the computational complexity of solving the model equations, however, the results presented here can be trivially extended to cases where P gx = P gy . 24 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintWe wish to stress that the work presented in this manuscript is an initial study on the effect of domain growth on spatial correlations between agents and, as such, we have chosen the simplest modelling framework for this. Therefore, an important consideration is whether the work presented here could be extended to other, perhaps more realistic models, such as an off-lattice IBM. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintIt is simpler to derive the pairwise density functions for GM2 for a domain of size (N x + 1) × (N y + 1) (this avoids having (N x − 1) in the denominator of a number of fractions). For GM2 it is also necessary to introduce some new notation. The distance of the lattice site m from the boundary (origin of growth) is m i and m j , i.e. the position of m, and the distance of the lattice site n from boundary, n i and n j , i.e. the position of m + (r x , r y ). The evolution of the pairwise density functions for the diagonal component of GM2 isNy j=1Ny j=1Ny j=1Ny j=131 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintThe terms on the RHS of Eq. Ny j=132 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint We rewrite Eq. (33) usingñ x ,m x andm x n x , which are constants defined asEqs. (34)-(36) can be evaluated directly. If we do so we obtain:We substitute Eqs. (34)-(36) into Eq. (33) to obtain 33 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintThat Eqs. (33) and (40) are equivalent can be shown by substituting Eqs. (34)-(36) into Eq. (40) . 34 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintIf we now apply the approximation,to Eq. (40) we obtain(r x , r y ; t)(r x , r y ; t)(r x , r y − 1; t)(r x , r y + 1; t)(r x , r y ; t)If we rearrange Eq. (42) and take the limit as δt → 0 we obtain dρ (Nx+1)×(Ny+1) 2(r x , r y ; t) dt = −(P gx (N x + 1) + P gy (N y + 1))ρ (Nx+1)×(Ny+1) 2(r x , r y ; t) + P gy N y −ñ y −m y + 2m y n y N y ρ (Nx+1)×(Ny+1) 2(r x , r y ; t)(r x , r y − 1; t)(r x , r y + 1; t)(r x , r y ; t)(r x − 1, r y ; t)35 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint which we can simplify further to obtain(r x , r y ; t)(r x , r y ; t)(r x , r y − 1; t)(r x , r y + 1; t)(r x − 1, r y ; t)If we now evaluate Eqs. (r x , r y ; t) dt = (45)(r x , r y ; t)(r x , r y ; t)(r x , r y − 1; t)(r x , r y + 1; t)(r x − 1, r y ; t) Ny j=1Ny j=1Ny j=12 (m, m + (r x , 0); t).(47) 37 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintWe rewrite Eq. (47) using Eqs. (48) 38 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprintWe now make the approximation Eq. (41) to obtain ρ (Nx+1)×(Ny+1) 2 (r x , 0; t + δt) = (1 − δtP gx (N x + 1) − δtP gy (N y + 1))ρ (Nx+1)×(Ny+1) 2 (r x , 0; t) + δtP gy m y n y N y ρ (Nx+1)×(Ny+1) 2(r x , 0; t) + δtP gy N y −ñ y −m y +m y n y N y ρ (Nx+1)×(Ny+1) 2(r x , 0; t) + δtP gy ñ y − 1 −m y n y N y +m y N y ρ (Nx+1)×(Ny+1) 2(r x , −1; t) + δtP gy m y − 1 −m y n y N y +ñ y N y ρ (Nx+1)×(Ny+1) 2(r x , −1; t)(r x − 1, 0; t)which we can simplify to obtain dρ (Nx+1)×(Ny+1) 2(r x , 0; t) dt = −P gx (r x + 1)ρ (Nx+1)×(Ny+1) 2(r x , 0; t)(r x − 1, 0; t) + P gy −(N y + 1) + 2m y n y N y + N y −ñ y −m y ρ (r x , 0; t) dt = −P gx (r x + 1)ρ (Nx+1)×(Ny+1) 2(r x , 0; t)(r x − 1, 0; t)(r x , 0; t)This is Eq. (20) in the main text.39 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint Appendix B: The evolution of the spatial correlations for GM1In Fig. 10 the evolution of the spatial correlations for GM1 can be seen. 40 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#We can access the date by index or by file name, but we have to change in the parse function\n",
    "#what kind of invoke we want\n",
    "print(parser.data_dicts[Dataset.BIORXIV][45])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Accesing certain elements of the paper"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[1mAbstract\u001b[0m\n",
      "Summary Statement: DUX4 is thought to mediate cytopathology through p53. Here, DUX4 is shown to kill primary myoblasts and promote pathological phenotypes in the iDUX4[2.7] mouse model on the p53-null background, calling into question this notion.FSHD is a genetically dominant myopathy caused by mutations that cause expression of the normally silent DUX4 gene. This transcription factor has been shown to interfere with myogenesis when misexpressed at very low levels in myoblasts, and to cause cell death when overexpressed at high levels. A previous report using adeno-associated virus to deliver high levels of DUX4 to mouse skeletal muscle demonstrated severe pathology that was suppressed on a p53 knockout background, implying that DUX4 acted through the p53 pathway. Here, we investigate the p53-dependence of DUX4 using both in vitro cellular and the transgenic iDUX4[2.7] mouse models. We find that inhibiting p53 has no effect on the cytoxicity of DUX4 in vitro. When crossed onto the p53 null background, we find no suppression of the male-specific lethality or skin phenotypes of the DUX4 transgene, and find that primary myoblasts from this mouse are still killed by DUX4 expression. These data challenge the notion that the p53 pathway is central to the pathogenicity of DUX4.166 words. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The exported cases of 2019 novel coronavirus infection that were confirmed outside of China provide an opportunity to estimate the cumulative incidence and confirmed case fatality risk (cCFR) in mainland China. Knowledge of the cCFR is critical to characterize the severity and understand the pandemic potential of COVID-19 in the early stage of the epidemic. Using the exponential growth rate of the incidence, the present study statistically estimated the cCFR and the basic reproduction number-the average number of secondary cases generated by a single primary case in a naïve population. We modeled epidemic growth either from a single index case with illness onset on 8 December, 2019 (Scenario 1), or using the growth rate fitted along with the other parameters (Scenario 2) based on data from 20 exported cases reported by 24 January, 2020. The cumulative incidence in China by 24 January was estimated at 6924 cases (95% CI: 4885, 9211) and 19,289 cases (95% CI: 10,901, 30,158), respectively. The latest estimated values of the cCFR were 5.3% (95% CI: 3.5%, 7.5%) for Scenario 1 and 8.4% (95% CI: 5.3%, 12.3%) for Scenario 2. The basic reproduction number was estimated to be 2.1 (95% CI: 2.0, 2.2) and 3.2 (95% CI: 2.7, 3.7) for Scenarios 1 and 2, respectively. Based on these results, we argued that the current COVID-19 epidemic has a substantial potential for causing a pandemic. The proposed approach provides insights in early risk assessment using publicly available data.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The World Health Organization (WHO) has declared the 2019 novel coronavirus (2019-nCoV) infection outbreak a global health emergency. Currently, there is no effective anti-2019-nCoV medication. The sequence identity of the 3CL proteases of 2019-nCoV and SARS is 96%, which provides a sound foundation for structural-based drug repositioning (SBDR). Based on a SARS 3CL protease X-ray crystal structure, we construct a 3D homology structure of 2019-nCoV 3CL protease. Based on this structure and existing experimental datasets for SARS 3CL protease inhibitors, we develop an SBDR model based on machine learning and mathematics to screen 1465 drugs in the DrugBank that have been approved by the U.S. Food and Drug Administration (FDA). We found that many FDA approved drugs are potentially highly potent to 2019-nCoV.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Accurate and rapid diagnosis of COVID-19 suspected cases plays a crucial role in timely quarantine and medical treatment. Developing a deep learning-based model for automatic COVID-19 detection on chest CT is helpful to counter the outbreak of SARS-CoV-2. A weakly-supervised deep learning-based software system was developed using 3D CT volumes to detect COVID-19. For each patient, the lung region was segmented using a pre-trained UNet; then the segmented 3D lung region was fed into a 3D deep neural network to predict the probability of COVID-19 infectious. 499 CT volumes collected from Dec. 13, 2019, to Jan. 23, 2020, were used for training and 131 CT volumes collected from Jan 24, 2020, to Feb 6, 2020, were used for testing. The deep learning algorithm obtained 0.959 ROC AUC and 0.976 PR AUC. There was an operating point with 0.907 sensitivity and 0.911 specificity in the ROC curve. When using a probability threshold of 0.5 to classify COVID-positive and COVID-negative, the algorithm obtained an accuracy of 0.901, a positive predictive value of 0.840 and a very high negative predictive value of 0.982. The algorithm took only 1.93 seconds to process a single patient's CT volume using a dedicated GPU. Our weaklysupervised deep learning model can accurately predict the COVID-19 infectious probability in chest CT volumes without the need for annotating the lesions for training. The easily-trained and highperformance deep learning algorithm provides a fast way to identify COVID-19 patients, which is beneficial to control the outbreak of SARS-CoV-2. The developed deep learning software is available at https://github.com/sydney0zq/covid-19-detection.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Artificial nanoparticles accumulate a protein corona layer in biological fluids, which significantly influences their bioactivity. As nanosized obligate intracellular parasites, viruses share many biophysical properties with artificial nanoparticles in extracellular environments and here we show that respiratory syncytial virus (RSV) and herpes simplex virus 1 (HSV-1) accumulate a rich and distinctive protein corona in different biological fluids. Moreover, we show that corona pre-coating differentially affects viral infectivity and immune cell activation. Additionally, we demonstrate that viruses bind amyloidogenic peptides in their corona and catalyze amyloid formation via surface-assisted heterogeneous nucleation. Importantly, we show that HSV-1 catalyzes the aggregation of the amyloid beta peptide (Aβ42), a major constituent of amyloid plaques in Alzheimer's disease, in-vitro and in animal models. Our results highlight the viral protein corona as an acquired structural layer that is critical for viral-host interactions and illustrate a mechanistic convergence between viral and amyloid pathologies. doi: bioRxiv preprint HSV-1 is an example of another virus with high prevalence, infecting nearly 70% of the human population 17 .HSV-1 is a double-stranded DNA virus consisting of an icosahedral nucleocapsid surrounded by tegument and envelope with virion sizes ranging from 155 to 240 nm 18 . HSV-1 is a neurotropic virus that infects peripheral . CC-BY-NC-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/246785 doi: bioRxiv preprint 4 sensory neurons and establishes latency 19 . Latent HSV-1 is occasionally reactivated causing peripheral pathology and under certain circumstances it can migrate into the central nervous system causing herpes simplex encephalitis; the most common cause of sporadic fatal viral encephalitis 19 . In the context of the current work we focused on the presumptive role of HSV-1 in the pathology of AD. A number of risk factors have been associated with AD, including the E4 allele of the apolipoprotein E (Apo-E), diabetes, vascular pathology, neuroinflammation and infections 20 . Several recent studies have supported the theory of a significant role of HSV-1 in the disease 21 . HSV-1 DNA was found to be localized within amyloid plaques in AD patients and HSV-1 infection has been shown to promote neurotoxic Aβ accumulation in human neural cells and to the formation of Aβ deposits in the brains of infected mice 22,23 . Moreover, the presence of anti-HSV IgM antibodies, which indicate HSV reactivation, is correlated with a high risk of AD and antiherpetic treatment is correlated with a reduced risk of developing dementia 24,25 . Despite these correlations, the mechanism by which viruses induce amyloid aggregation, the major pathological hallmark of AD, is not known.In the present study, we demonstrated that upon encountering different biological fluids, RSV accumulated extensive and distinct protein coronae compared to HSV-1 and synthetic liposomes. The various coronae were dependent on the biological fluid and exerted markedly different effects on RSV infectivity and capacity to activate monocyte-derived dendritic cells (moDCs). Moreover, upon interaction with an amyloidogenic peptide derived from IAPP, RSV accelerated the process of amyloid aggregation via surface-assisted heterogenous nucleation.This amyloid catalysis was also demonstrated for HSV-1 and the Aβ42 peptide in-vitro and in an AD animal model.Our findings highlight the importance of viral protein corona interactions for viral pathogenesis and provide a direct mechanistic link between viral and amyloid pathologies.ResultsBased on the extensive literature describing the significant role of corona factors in synthetic nanoparticle functionality, we used established techniques to answer questions regarding RSV pathogenicity 26 . Using proteomics, we assessed the RSV protein corona profiles in adult human plasma (HP), juvenile (6 months old infants tested RSV negative at the time of sample collection) human plasma (jHP), human bronchoalveolar lavage fluid (BALF) from healthy adults, rhesus macaque plasma (MP), and fetal bovine serum (FBS). These biological fluids represent different microenvironments encountered by the virus in terms of tissue tropism (HP vs. BALF), zoonosis (MP) and culturing conditions (FBS). The biological fluids were screened for antibodies against RSV using ELISA, and both adult HP and BALF contained high levels of anti-RSV IgG antibodies, unlike jHP, MP and FBS (Supplementary Fig. 1 ). Viral stocks were produced in serum-free conditions to prevent initial contamination with FBS proteins. Virions produced under serum-free conditions were incubated with 10% v/v solutions of different biological fluids.Controls included non-infected cell medium representing the background cellular secretome, and synthetic lipid vesicles of a size comparable to RSV (200 nm) with positively or negatively charged surfaces. In addition, we compared the coronae of RSV to HSV-1 to probe for differences between different viruses of relatively similar size. After incubation for 1 h at 37 ˚C, the virions were re-harvested by centrifugation and washed twice before performing mass spectrometry-based proteomic analyses.Six replicates from two independent experiments. Statistics were calculated using GraphPad Prism 7 software.The detection of specific anti-RSV IgG antibodies in biological fluids was performed using Human Anti-Respiratory syncytial virus IgG ELISA Kit (ab108765, abcam®, Sweden), according to manufacturer's protocol.All biological fluids (FBS, HP, MP, BALF) were diluted to a protein concentration of 0.3 mg/ml before performing the assay and results were compared to the positive, cutoff, and negative controls provided by the kit manufacturer.HEp-2 cells were seeded in maintenance medium until they reached 50-60% confluency. On the day of infection, the cells were washed twice with warm PBS and medium was changed to VP-SFM with gentamycin. Before adding to the cells, sucrose cushion-concentrated RSV stocks were pre-incubated with different biological fluids (FBS, HP, MP, BALF) at a final protein concentration of 0.3 mg/ml in VP-SFM with 10 mM sodium citrate for 1 h at 37 ˚C. The corona-precoated viruses were then added to the cells in serum-free conditions (diluted 10x) at a MOI of 1. 24h post-infection, the medium was changed back to growth medium and cells were visualized with fluorescence microscopy 72h post-infection. After visualization, the cells were washed and stained with LIVE/DEAD™ Fixable Far Red Dead Cell Stain Kit (ThermoFisher, USA) for 30 min at 4 ˚C in Dulbecco's phosphate-buffered saline (DPBS no calcium, no magnesium, ThermoFisher), then fixed and washed for flow cytometry using Cytofix/Cytoperm™ (BD, USA) according to manufacturer's protocol. Cells were then scraped and resuspended in DPBS, and the data was acquired using MACSQuant® Analyser 10 flowcytometer (Miltenyi Biotec, Sweden). The data was analyzed by FlowJo software (TreeStar) by excluding the dead cells stained with far-red fluorescent dye and subsequent calculation of GFP positive cells within the viable cell population. For experiments with palivizumab, cells were treated with different concentrations of the antibody before infection.Human monocytes were negatively selected from buffy coats using the RosetteSep Monocyte Enrichment Kit (1mL/10mL buffy coat; StemCell Technologies) and differentiated into moDC, using GM-CSF (250ng/mL; PeproTech) and IL-4 (6.5ng/mL; R&D Systems) for 6 days in 37°C, 5% CO2 at a density of 5x10 5 cells/mL in RPMI 1640 completed with 10% FCS, 1mM sodium pyruvate, 10mM HEPES, 2mM L-glutamine, and 1% and penicillin/streptomycin (ThermoFisher, USA). Immature moDC were exposed to RSV pre-incubated with different biological fluids for 4h in serum free media, washed and then incubated in serum containing medium for 72 h before analyses of CD86 and RSV-GFP expression. Dead cells were excluded using Live/Dead fixable near-IR dead cell stain kit (ThermoFisher). Flow cytometry sample data were acquired on a Fortessa (BD Biosciences) and the analysis was performed in FlowJo software (TreeStar).NNFGAIL, GNNQQNY, Aβ42, Aβ40 were synthesized and purified by Pepscan (The Netherlands) and the final molecular weight was verified by MS. 10 µl of dimethyl sulfoxide (DMSO) (ThermoFisher, USA) were added to .\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Sodium-bile acid cotransporter, also denominated sodium-taurocholate cotransporting polypeptide (NTCP) is an integral membrane protein with multiple hydrophobic transmembrane domains. The third extracellular domain of NTCP presents a stretch of nine aminoacids (KGIVISLVL) that is characterized by pronounced hydrophobicity and serves as receptor for a protein, preS1, showing the hydrophobic epta-peptide sequence NPLGFFP. Vitamin D-binding protein macrophage activating factor (DBP-MAF) is a multifunctional protein that is characterized by two hydrophobic regions able to bind fatty acids and vitamin D, respectively. Here we demonstrate that NTCP and DBP-MAF show significant sequence similarities as far as hydrophobic stretches of aminoacids are concerned. Alignment of the sequence of seven aminoacids preceding the 157-KGIVISLVL-165 stretch of NTCP shows four aminoacids that are identical to those of the corresponding sequence of DBP-MAF, and two that are conserved substitutions. In addition, in the sequence of DBP-MAF that is aligned with the sequence YKGIVISLVL of NTCP, there are two contiguous negatively charged aminoacids (ED) and, in the preceding epta-peptide sequence, there are three negatively charged aminoacids (D-ED), whereas in the corresponding sequence of NTCP there are only two (D--D) that are not contiguous. This concentration of negatively charged aminoacids may be involved in binding of protein inserts characterized by high density of positively charges residues. The alternating hydrophobic and electrostatic interactions described in this paper may help elucidating the biological roles of these proteins as far as protein-protein interactions are concerned.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Recently emerged Wuhan's novel coronavirus designated as SARS-CoV-2, a causative agent of coronavirus disease 2019 (COVID-19) is rapidly spreading its pathogenicity throughout the world now. More than 4000 mortalities have occurred worldwide till the writing of this article and this number is increasing every passing hour. World Health Organization (WHO) has declared it as a global public health emergency. The multiple sequence alignment data correlated with already published reports on SARS-CoV-2 indicated that it is closely related to Bat-Severe Acute Respiratory Syndrome like coronavirus (Bat CoV SARS-like) and wellstudied Human SARS. In this study, we have exploited the complementary approach to examine the intrinsically disordered regions in proteome of SARS-CoV-2 using Bat SARSlike and Human SARS CoVs as comparative models. According to our findings, SARS-CoV-2 proteome contains a significant amount of ordered proteins except Nucleocapsid, Nsp8, and ORF6. Further, cleavage sites in replicase 1ab polyprotein are found to be highly disordered. We have extensively investigated the dark proteome in SARS-CoV-2 which will have implications for the structured and unstructured biology of SARS or SARS-like coronaviruses.Significance: The infection caused by Novel Coronavirus (SARS-CoV-2) is responsible for the current pandemic that cause severe respiratory disease and pneumonia-like infection in humans. Currently, there is no such in-depth information on protein structure and function available in public domain and no effective anti-viral or vaccines are available for the treatment of this infection. Our study provides comparative order and disorder-based proteome information with Human SARS and Bat CoV that may be useful for structure-based drug discovery.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Genome Detective is a web-based, user-friendly software application to quickly and accurately assemble all known virus genomes from next generation sequencing datasets. This application allows the identification of phylogenetic clusters and genotypes from assembled genomes in FASTA format. Since its release in 2019, we have produced a number of typing tools for emergent viruses that have caused large outbreaks, such as Zika and Yellow Fever Virus in Brazil. Here, we present The Genome Detective Coronavirus Typing Tool that can accurately identify novel coronavirus (2019-nCoV) sequences isolated in China and around the world. The tool can accept up to 2,000 sequences per submission and the analysis of a new whole genome sequence will take approximately one minute. The tool has been tested and validated with hundreds of whole genomes from ten coronavirus species, and correctly classified all of the SARS-related coronavirus (SARSr-CoV) and all of the available public data for 2019-nCoV. The tool also allows tracking of new viral mutations as the outbreak expands globally, which may help to accelerate the development of novel diagnostics, drugs and vaccines.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "medRxiv preprint Funding Information: Health Commission of Hubei Province scientific research project (No. WJ2019Q048 to M.Z.)We declare no conflict of interest.Abstract: Since SARS-CoV-2 infection was first identified in December 2019, it spread rapidly and a global pandemic of COVID-19 has occurred. ACE2, the receptor for entry into the target cells by SARS-CoV-2, was found to abundantly express in testes, including spermatogonia, Leydig and Sertoli cells. However, there is no clinical evidence about whether SARS-CoV-2 infection can affect male gonadal function so far.In this study, we compared the sex-related hormones between 81 reproductive-aged men with SARS-CoV-2 infection and 100 age-matched healthy men, and found that serum luteinizing hormone (LH) was significantly increased, but the ratio of testosterone (T) to LH and the ratio of follicle stimulating hormone (FSH) to LH were dramatically decreased in males with COVID-19. Besides, multivariable regression analysis indicated that c-reactive protein (CRP) level was significantly associated with serum T:LH ratio in COVID-19 patients. This study provides the first direct evidence about the influence of medical condition of COVID-19 on male sex hormones, alerting more attention to gonadal function evaluation among patients recovered from SARS-CoV-2 infection, especially the reproductive-aged men.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "ZG and YL made substantial contributions to the study concept and design. LW was in charge of the manuscript draft. XL, CH, SY and DL took responsibility for obtaining written consent from patients, obtaining ethical approval, collecting samples, and confirming data accuracy. LW and ZG participated in drafting the manuscript, and revising it on the basis of reviewers' comments. LW made substantial contributions to data acquisition, analysis, and interpretation.We declare no competing interests.With the permission of the corresponding authors, we can provide participant data without names and identifiers, but not the study protocol, statistical analysis plan, or informed consent form. Data can be provided after the Article is published. Once the data can be made public, the research team will provide an email address for communication. The corresponding authors have the right to decide whether to share the data or not based on the research objectives and plan provided.All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "doi: bioRxiv preprint 106 components. ALLAN and MET, by contrast, are more variable and very close to each other and to 107 untreated control (Fig. 2b, Extended data Fig. 4c). 108 109 Drug synergies and lifespan extension 110 We next determine effect of different drug combinations on lifespan in nematodes. We began with 111 MET and RAP due to their translational importance in humans 14,27 . However, RAP combined with 112 MET at their optimal doses did not show any further benefit on lifespan (Extended data Fig. 2j,p). We 113 carried out additional trials, exploring all possible combinations of optimal and half-optimal dose for 114 MET and RAP (Extended data Fig. 2j-p). Combination of RAP and MET each at their half-optimal 115 dose result in further extension in maximum lifespan (Fig. 2e, Extended data Fig. 2m,s) but did not 116 result in further mean lifespan extension (Extended data 2p). These weak benefits of combining MET 117 with RAP are consistent with results in mice where the existing evidence suggests that addition of Figures S3 lifespan comparison of triple combinations supplementary for figure 2. a, RAP+RIF+PSORA did not show further benefit compared to the dual synergistic combinations RAP+RIF and RIF+PSORA. b, RAP+RIF+MET is toxic compared to RAP+RIF. More than 50 worms per condition.. c, RAP+PSORA+ALLAN did not show further benefit compared to single drug constituents. d, RAP+RIF+ALLAN and RIF+PSORA+ALLAN resulted in a significant lifespan extension compared to RAP+RIF and RIF+PSORA respectively. log-rank with proper adjustment. e, Mean and f, Maximum lifespan of figure d showed a monotonic increase from single drugs to double and triple combinations. One-way ANOVA, Bonferroni multiple comparison *doi: bioRxiv preprint 235 12 Ejsing, C. S. et al. Collision-induced dissociation pathways of yeast sphingolipids and their 236 molecular profiling in total lipid extracts: a study by quadrupole TOF and linear ion trap-237 orbitrap mass spectrometry.\n",
      "\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "300 words) 33 Quantification of aerosolized influenza virus is used for determining inhalation exposure. Several 34 bioaerosol samplers and analytical methods have been used; however, the detection and 35 quantification of influenza virus among aerosol samples remains challenging. Therefore, 36 improved viral aerosol measurement methods are needed. This study evaluated influenza virus 37 recovery among three filter types polytetrafluoroethylene, polyvinylchloride and polystyrene. 38 Polytetrafluoroethylene, polyvinylchloride are fabricated filter materials and commonly used in 39 the scientific literature to sample for viral aerosols. A novel, electrospun polystyrene filter 40 material may improve viral aerosol recovery during filter-based air sampling. The filter materials 41 were compared across the following conditions: treated with or without air, filter wash buffer 42 (HBSS or PBS), and viral RNA extraction method (QIAamp Viral RNA Mini Kit or Trizol). 43 Twenty trials were completed in a chamber and samples were analyzed using RT-qPCR. Viral 44 recovery was significantly different (p-value < .0001) by filter type. Polystyrene filter use 45 resulted in recovery of the most viral RNA. Air sampling did not affect the recovery of viral 46 RNA from the filter materials (p-values > 0.05). Viral RNA concentrations were significantly 47 different across extraction methods for all comparisons (p-values < 0.05). Our results 48 demonstrated that the novel polystyrene filter material resulted in the highest concentration of 49 extracted RNA compared to the commonly used polytetrafluoroethylene and polyvinylchloride, 50 which we speculate may be related to the chemical composition of the filter material (e.g., 51 polystyrene is an aromatic hydrocarbon whereas polytetrafluoroethylene and polyvinylchloride 52 contain more polar, and thus potentially reactive, carbon-halogen bonds). Air sampling did not 53 have an effect on viral RNA recovery. Using Hanks Balanced Salt Solution with QIAamp Viral . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/441154 doi: bioRxiv preprint 54 RNA Mini Kit, and Phosphate-buffered saline with the Trizol extraction, resulted in the most 55 viral RNA recovery. 56 57 58\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Since COVID-19 emerged in early December, 2019 in Wuhan and swept across China Mainland, a series of large-scale public health interventions, especially Wuhan lock-down combined with nationwide traffic restrictions and Stay At Home Movement, have been taken by the government to control the epidemic. Based onBaidu Migration data and the confirmed cases data, we identified two key factors affecting the later (e.g February 27, 2020) cumulative confirmed cases in non-Wuhan region (y). One is the sum travelers from Wuhan during January 20 to January 26 (x1), which had higher infected probability but lower transmission ability because the human-to-human transmission risk of COVID-19 was confirmed and announced on January 20. The other is the \"seed cases\" from Wuhan before January 19, which had higher transmission ability and could be represented with the confirmed cases before January 29 (x2) due to a mean 10-day delay between infection and detection. A simple yet effective regression model then was established as follow: y= 70.0916+0.0054×x1+2.3455×x2 (n = 44, R 2 = 0.9330, P<10 -7 ). Even the lock-down date only delay or in advance 3 days, the estimated confirmed cases by February 27 in non-Wuhan region will increase 35.21% or reduce 30.74% -48.59%. Although the above interventions greatly reduced the human mobility, Wuhan lock-down combined with nationwide traffic restrictions and Stay At Home Movement do have a determining effect on the ongoing spread of COVID-19 across China Mainland. The strategy adopted by China has changed the fast-rising curve of newly diagnosed cases, the international community should learn from lessons of Wuhan and experience from China. Efforts of 29 Provinces and 44 prefecture-level cities against COVID-19 were also assessed preliminarily according to the interpretive model. Big data has played and will continue playing an important role in public health.\n",
      "\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The detection of serum antibodies to the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is emerging as a new tool for the coronavirus disease-2019 diagnosis. Since many coronaviruses are sensitive to heat, heating inactivation of samples at 56 ℃ prior to testing is considered a possible method to reduce the risk of transmission, but the effect of heating on the measurement of SARS-CoV-2 antibodies is still unclear. By comparing the levels of SARS-CoV-2 antibodies before and after heat inactivation of serum at 56 ℃ for 30 minutes using a quantitative fluorescence immunochromatographic assay, we shown that heat inactivation significantly interferes with the levels of antibodies to SARS-CoV-2. The IgM levels of all the 34 serum samples (100%) from COVID-19 patients decreased by an average level of 53.56%. The IgG levels were decreased in 22 of 34 samples (64.71%) by an average level of 49.54%. Similar changes can also be observed in the non-COVID-19 diseases group (n=9). Of note, 44.12% of the detected IgM levels were dropped below the cut-off value after heating, suggesting heat inactivation can lead to false-negative results of these samples. Our results indicate .CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101//doi.org/10. /2020 that heat inactivation of serum at 56 ℃ for 30 minutes interferes with the immunoanalysis of antibodies to SARS-CoV-2. Heat inactivation prior to immunoanalysis is not recommended and the possibility of false-negative results should be considered if the sample was pre-inactivated by heating.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Clinical assessment of possible infection with SARS-CoV-2, the novel coronavirus responsible for the outbreak of COVID-19 respiratory illness, has been a major activity of infectious diseases services in the UK and elsewhere since the first report of cases in December 2019. We report our case series of 68 patients, reviewed by Infectious Diseases Consultants at a Regional Infectious Diseases Unit in the UK. We prospectively evaluated our service between the 29 th Jan 2020 and 24 th Feb 2020.Demographic, clinical, epidemiological and laboratory data were collected. We have compared clinical features and subsequent diagnosis between well patients not requiring admission for clinical reasons or antimicrobials with those assessed as needing either admission or antimicrobial treatment.A, RSV, non SARS/MERS coronaviruses, rhinovirus/enterovirus. 9/68 were treated with antimicrobials, 15/68 were admitted to a negative pressure room of whom 5/68 were admitted solely due to an inability to isolate at home. Patients requiring either admission on clinical grounds or antimicrobials (14/68) were similar to those not requiring admission or antimicrobials, with modestly more fever and shortness of breath in the clinically admitted / antimicrobial group. The most commonly prescribed antimicrobials were doxycycline, moxifloxacin and oseltamivir.The majority of patients had mild illness which did not require a clinical intervention to manage. This finding supports a community testing approach supported by clinicians to review the proportion of more unwell patients.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "12 The current classification of animal viruses primarily relates to the virus molecular world, the 13 genomic architecture and the corresponding host-cell infection cycle. This virus centered 14 perspective does not make allowance for the precept that virus fitness hinges on the virus 15 transmission success. Virus transmission reflects the infection-shedding-transmission 16 dynamics and, with it, the organ system involvement and other, macroscopic dimensions of 17 the host environment. This study examines the transmission ecology of the world main 18 livestock viruses, 36 in total, belonging to eleven different families, and a mix of RNA, DNA 19 and retroviruses. Viruses are virtually ranked in an outer-to inner-body fashion, based on 20 the shifting organ system involvement and associated infection-shedding-transmission 21 dynamics. As a next step, this ranking is disentangled with the aim to contrast two main host 22 45 the virus. Two opposing virus evolution pathways emerge, respectively for generalist type, 46 outer-body and for specialist, inner-body viruses.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Although bats are increasingly becoming the focus of scientific studies due to their unique properties, these exceptional animals are still among the least studied mammals. Assembly quality and completeness of bat genomes vary a lot and especially non-coding RNA (ncRNA) annotations are incomplete or simply missing. Accordingly, standard bioinformatics pipelines for gene expression analysis often ignore ncRNAs such as microRNAs or long antisense RNAs. The main cause of this problem is the use of incomplete genome annotations. We present a complete screening for ncRNAs within 16 bat genomes. NcRNAs affect a remarkable variety of vital biological functions, including gene expression regulation, RNA processing, RNA interference and, as recently described, regulatory processes in viral infections. Within all investigated bat assemblies we annotated 667 ncRNA families including 162 snoRNAs and 193 miRNAs as well as rRNAs, tRNAs, several snRNAs and lncRNAs, and other structural ncRNA elements. We validated our ncRNA candidates by six RNA-Seq data sets and show significant expression patterns that have never been described before in a bat species on such a large scale. Our annotations will be usable as a resource (Electronic Supplement) for deeper studying of bat evolution, ncRNAs repertoire, gene expression and regulation, ecology, and important host-virus interactions.Supplementary information: is available at rna.uni-jena.de/supplements/bats, the Open Science Framework (doi.org/10.17605/OSF.IO/4CMDN), and GitHub (github.com/rnajena/bats_ncrna).\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "With the ongoing outbreak of Coronavirus Disease 2019 , infected patients within and beyond the epidemic area, Wuhan, China, showed different epidemiological and clinical characteristics. There is a paucity of data concerning coinfection with other common respiratory pathogens in COVID-19 patients outside of Wuhan.We conducted a double-centre study recruiting 68 patients with severe acute respiratory coronavirus 2 (SARS-CoV-2) infection confirmed by nucleic acid testing in Qingdao and Wuhan from January 17 to February 16, 2020. Indirect immunofluorescence was performed to detect the specific IgM antibody against common respiratory pathogens in collected acute phase serum.: Of the 68 patients with SARS-CoV-2 infection, 30 (44.12%) were from Qingdao. The median age of Qingdao and Wuhan patients were 50 (IQR: 37-59) and 31 (IQR: 28-38) years, respectively, and the majority of patients were female in Qingdao (60.00%) and Wuhan (55.26%). Among COVID-19 patients in Qingdao, 24 (80.00%) of them had IgM antibodies against at least one respiratory pathogen, whereas only one (2.63%) of the patients in Wuhan had positive results for serum IgM antibody detection (P<0.0001). The most common respiratory pathogens detected in Qingdao COVID-19 patients were influenza virus A (60.00%) and influenza virus B (53.33%), followed by mycoplasma pneumoniae (23.33%) and legionella pneumophila (20.00%). While the pattern for coinfection in patients with community-acquired pneumonia in Qingdao was quite different, with a positive rate of only 20.90%. Interpretation: We reported a large proportion of COVID-19 patients with coinfection of seasonal respiratory pathogens in Qingdao, northeast China, which differed greatly from the patients in Wuhan, central China. Precautions are needed when dealing with COVID-19patients beyond the epidemic centre who have coinfection with other respiratory pathogens.We highly recommend adding SARS-CoV-2 to routine diagnostic testing in capable hospitals to prevent misdetection of the virus.\n",
      "\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The outbreak of the novel coronavirus disease, COVID-19, originating from Wuhan, China in early December, has infected more than 70,000 people in China and other countries and has caused more than 2,000 deaths. As the disease continues to spread, the biomedical society urgently began identifying effective approaches to prevent further outbreaks. Through rigorous epidemiological analysis, we characterized the fast transmission of COVID-19 with a basic reproductive number 5.6 and proved a sole zoonotic source to originate in Wuhan. No changes in transmission have been noted across generations. By evaluating different control strategies through predictive modeling and Monte carlo simulations, a comprehensive quarantine in hospitals and quarantine stations has been found to be the most effective approach. Government action to immediately enforce this quarantine is highly recommended.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Public health preparedness for coronavirus disease 2019 is challenging in the absence of setting-specific epidemiological data. Here we describe the epidemiology of seasonal human coronaviruses (sCoVs) and other cocirculating viruses in the West of Scotland, UK. We analyzed routine diagnostic data for >70,000 episodes of respiratory illness tested molecularly for multiple respiratory viruses between 2005 and 2017. Statistical associations with patient age and sex differed between CoV-229E, CoV-OC43 and CoV-NL63. Furthermore, the timing and magnitude of sCoV outbreaks did not occur concurrently and coinfections were not reported. With respect to other cocirculating respiratory viruses, we found evidence of positive, rather than negative, interactions with sCoVs. These findings highlight the importance of considering cocirculating viruses in the differential diagnosis of COVID-19. Further work is needed to establish the occurrence/degree of cross-protective immunity conferred across sCoVs and with COVID-19, as well as the role of viral coinfection in COVID-19 disease severity.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Rapid screening and low-cost diagnosis play a crucial role in choosing the correct course of intervention e.g., drug therapy, quarantine, no action etc. when dealing with highly infectious pathogens. This is especially important if the disease-causing agent has no effective treatment, such as the novel coronavirus SARS-CoV-2 (the pathogen causing COVID-19), and shows no or similar symptoms to other common infections. We report a silicon-based integrated Point-of-Need (PoN) transducer (TriSilix) that can chemically-amplify and detect pathogen-specific sequences of nucleic acids (NA) quantitatively in real-time. Unlike other silicon-based technologies, TriSilix can be produced at wafer-scale in a standard laboratory; we have developed a series of methodologies based on metal-assisted chemical (wet) etching, electroplating, thermal bonding and laser-cutting to enable a cleanroom-free low-cost fabrication that does not require processing in an advanced semiconductor foundry. TriSilix is, therefore, resilient to disruptions in the global supply chain as the devices can be produced anywhere in the world. To create an ultra-low-cost device, the architecture proposed exploits the intrinsic properties of silicon and integrates three modes of operation in a single chip: i) electrical (Joule) heater, ii) temperature sensor (i.e. thermistor) with a negative temperature coefficient that can provide the precise temperature of the sample solution during reaction and iii) electrochemical sensor for detecting target NA. Using TriSilix, the sample solution can be maintained at a single, specific temperature (needed for isothermal amplification of NA such as Recombinase Polymerase Amplification (RPA) or cycled between different temperatures (with a precision of ±1.3 °C) for Polymerase Chain Reaction (PCR) while the exact concentration of amplicons is measured quantitatively and in real-time electrochemically. A single 4-inch Si wafer yields 37 TriSilix chips of 10×10×0.65 mm in size and can be produced in 7 hours, costing ~US $0.35 per device. The system is operated digitally, portable and low powercapable of running up to 35 tests with a 4000 mAh battery (a typical battery capacity of a modern smartphone). We were able to quantitatively detect a 563-bp fragment (Insertion Sequence IS900) of the genomic DNA of M. avium subsp.paratuberculosis (extracted from cultured field samples) through PCR in real-time with a Limit-of-author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "As the predominant etiological agent of the common cold, human rhinovirus (HRV) is 19 the leading cause of human infectious disease. Early studies showed monovalent formalin-20 inactivated HRV vaccine can be protective, and virus-neutralizing antibodies (nAb) correlated 21 with protection. However, co-circulation of many HRV types discouraged further vaccine 22 efforts. We approached this problem straightforwardly. We tested the hypothesis that increasing 23 virus input titers in polyvalent inactivated HRV vaccine will result in broad nAb responses. Here, 24 we show that serum nAb against many rhinovirus types can be induced by polyvalent, 25 inactivated HRVs plus alhydrogel (alum) adjuvant. Using formulations up to 25-valent in mice 26 and 50-valent in rhesus macaques, HRV vaccine immunogenicity was related to sufficient 27 quantity of input antigens, and valency was not a major factor for potency or breadth of the 28 response. We for the first time generated a vaccine capable of inducing nAb responses to 29 numerous and diverse HRV types.30 31 38\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "We have developed a generalizable 'smart molecular diagnostic' capable of accurate point-ofcare (POC) detection of variable nucleic acid targets. Our one-pot isothermal assay relies on multiplex execution of four loop-mediated isothermal amplification reactions, with primers that are degenerate and redundant, thereby increasing the breadth of targets while reducing the probability of amplification failure. An easy-to-read visual answer is computed directly by a multi-input Boolean OR gate signal transducer that uses degenerate strand exchange probes to assess any combination of amplicons. We demonstrate our platform by using the same assay to detect divergent Asian and African lineages of the evolving Zika virus (ZIKV), while maintaining selectivity against non-target viruses. Direct analysis of biological specimens proved possible, with 20 virions / µl being directly detected in human saliva within 90 minutes, and crudely macerated ZIKV-infected Aedes aegypti mosquitoes being identified with 100% specificity and sensitivity.The ease-of-use with minimal instrumentation, broad programmability, and built-in fail-safe reliability make our smart molecular diagnostic attractive for POC use.While point-of-care (POC) diagnostic assays can be performed at or near the site of sample acquisition, they have for the most part been considered to be relatively simplistic tests that provide relatively little information to a clinician or public health worker. Familiar examples include electrochemical sensors for glucose, 1 or rapid immunoassays for metabolites such as human chorionic gonadotropin (the canonical pregnancy test), 2 and pathogens, either directly (influenza viruses) or via immune responses (antibodies against HIV-1/2). 3 The range of conditions and pathogens that could be tested for could likely be greatly expanded by developing POC diagnostics for nucleic acids 3 that would have greater sensitivity and accuracy. However, the current gold standard for molecular diagnostics, the quantitative polymerase chain reaction (qPCR), requires significant technical expertise and expensive and cumbersome equipment. Even portable instruments, such the Cepheid GeneXpert Omni, cost several thousand dollars and rely on expensive qPCR cartridge consumables for individual tests.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The human coronavirus HCoV-19 infection can cause acute respiratory distress syndrome 47 (ARDS), hypercoagulability, hypertension, extrapulmonary multiorgan dysfunction. Effective 48 antiviral and anti-coagulation agents with safe clinical profiles are urgently needed to improve the 49 overall prognosis. We screened an FDA approved drug library and found that an anticoagulant agent 50 dipyridamole (DIP) suppressed HCoV-19 replication at an EC50 of 100 nM in vitro. It also elicited 51 potent type I interferon responses and ameliorated lung pathology in a viral pneumonia model. In 52 analysis of twelve HCoV-19 infected patients with prophylactic anti-coagulation therapy, we found 53 that DIP supplementation was associated with significantly increased platelet and lymphocyte counts 54 and decreased D-dimer levels in comparison to control patients. Two weeks after initiation of DIP 55 treatment, 3 of the 6 severe cases (60%) and all 4 of the mild cases (100%) were discharged from the 56 hospital. One critically ill patient with extremely high levels of D-dimer and lymphopenia at the time 57 of receiving DIP passed away. All other patients were in clinical remission. In summary, HCoV-19 58 infected patients could potentially benefit from DIP adjunctive therapy by reducing viral replication, 59 suppressing hypercoagulability and enhancing immune recovery. Larger scale clinical trials of DIP 60 are needed to validate these therapeutic effects.61 62\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Objective: This paper studied the relationship between personality traits and mental health conditions of medical personnel to provide a basis and reference for the implementation of targeted education on mental health.A self-report inventory, the Symptom Checklist-90 (SCL-90), was used to investigate the mental health status of 548 medical personnel dealing with the new coronavirus pneumonia in eight provinces and cities of China.The overall mean SCL-90 score and mean values of factors (somatization, obsessive-compulsive, anxiety, phobic anxiety, and psychoticism) of the medical personnel were significantly higher than in the norm group (p < 0.05), while their average interpersonal sensitivity score was significantly lower (p < 0.01). In addition, personal factors affecting the mental health status of medical personnel were identified.( all p < 0.05).The overall mental health status of medical personnel responding to new coronavirus pneumonia is generally higher than that of the norm group in China. The results of this study should contribute to measures to alleviate the psychological pressures on medical personnel dealing with the new coronavirus epidemic in China.[Keywords] medical personnel; new coronavirus pneumonia; SCL-90 inventory; mental health; influencing factors All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The COVID-19 outbreak containment strategies in China based on non-pharmaceutical interventions (NPIs) appear to be effective. Quantitative research is still needed however to assess the efficacy of different candidate NPIs and their timings to guide ongoing and future responses to epidemics of this emerging disease across the World.We built a travel network-based susceptible-exposed-infectious-removed (SEIR) model to simulate the outbreak across cities in mainland China. We used epidemiological parameters estimated for the early stage of outbreak in Wuhan to parameterise the transmission before NPIs were implemented. To quantify the relative effect of various NPIs, daily changes of delay from illness onset to the first reported case in each county were used as a proxy for the improvement of case identification and isolation across the outbreak. Historical and near-real time human movement data, obtained from Baidu location-based service, were used to derive the intensity of travel restrictions and contact reductions across China. The model and outputs were validated using daily reported case numbers, with a series of sensitivity analyses conducted.We estimated that there were a total of 114,325 COVID-19 cases (interquartile range [IQR] 76,776 -164,576) in mainland China as of February 29, 2020, and these were highly correlated (p<0.001, R 2 =0.86) with reported incidence.Without NPIs, the number of COVID-19 cases would likely have shown a 67-fold increase (IQR: 44 -94), with the effectiveness of different interventions varying. The early detection and isolation of cases was estimated to prevent more infections than travel restrictions and contact reductions, but integrated NPIs would achieve the strongest and most rapid effect. If NPIs could have been conducted one week, two weeks, or three weeks earlier in China, cases could have been reduced by 66%, 86%, and 95%, respectively, together with significantly reducing the number of affected areas. However, if NPIs were conducted one week, two weeks, or three weeks later, the number of cases . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.is the (which was not peer-reviewed) The copyright holder for this preprint .\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The appearance of the novel betacoronavirus SARS-CoV-2 represents a major threat to human health, and its diffusion around the world is causing dramatic consequences. The knowledge of the 3D structures of SARS-CoV-2 proteins can facilitate the development of therapeutic and diagnostic molecules. Specifically, comparative analyses of the structures of SARS-CoV-2 proteins and homologous proteins from previously characterized viruses, such as SARS-CoV, can reveal the common and/or distinctive traits that underlie the mechanisms of recognition of cell receptors and of molecules of the immune system.Herein, we apply our recently developed energy-based methods for the prediction of antibodybinding epitopes and protein-protein interaction regions to the Receptor Binding Domain (RBD) of the Spike proteins from SARS-CoV-2 and SARS-CoV. Our analysis focusses only on the study of the structure of RBDs in isolation, without making use of any previous knowledge of binding properties. Importantly, our results highlight structural and sequence differences among the regions that are predicted to be immunoreactive and bind/elicit antibodies. These results provide a rational basis to the observation that several SARS-CoV RDB-specific monoclonal antibodies fail to appreciably bind the SARS-CoV-2 counterpart. Furthermore, we correctly identify the region of SARS-CoV-2 RBD that is engaged by the cell receptor ACE2 during viral entry into host cells.The data, sequences and structures we present here can be useful for the development of novel therapeutic and diagnostic interventions.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "A new coronavirus, SARS-CoV-2, has recently emerged to cause a human pandemic.Whereas molecular diagnostic tests were rapidly developed, serologic assays are still lacking, yet urgently needed. Validated serologic assays are important for contact tracing, identifying the viral reservoir and epidemiological studies. Here, we developed serological assays for the detection of SARS-CoV-2 neutralizing, spike-and nucleocapsid-specific antibodies. Using serum samples from patients with PCR-confirmed infections of SARS-CoV-2, other coronaviruses, or other respiratory pathogenic infections, we validated and tested various antigens in different in-house and commercial ELISAs. We demonstrate that most PCRconfirmed SARS-CoV-2 infected individuals seroconverted, as revealed by sensitive and specific in-house ELISAs. We found that commercial S1 IgG or IgA ELISAs were of lower specificity while sensitivity varied between the two, with IgA showing higher sensitivity. Overall, the validated assays described here can be instrumental for the detection of SARS-CoV-2-specific antibodies for diagnostic, seroepidemiological and vaccine evaluation studies. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "To extend the frontier of genome editing and enable the radical redesign of mammalian genomes, we developed a set of dead-Cas9 base editor (dBE) variants that allow editing at tens of thousands of loci per cell by overcoming the cell death associated with DNA double-strand breaks (DSBs) and single-strand breaks (SSBs). We used a set of gRNAs targeting repetitive elementsranging in target copy number from about 31 to 124,000 per cell. dBEs enabled survival after 5 large-scale base editing, allowing targeted mutations at up to ~13,200 and ~2610 loci in 293T and human induced pluripotent stem cells (hiPSCs), respectively, three orders of magnitude greater than previously recorded. These dBEs can overcome current on-target mutation and toxicity barriers that prevent cell survival after large-scale genome engineering.One Sentence Summary: Base editing with reduced DNA nicking allows for the simultaneous 10\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The spread of the novel coronavirus (SARS-CoV-2) has triggered a global emergency, that demands urgent solutions for detection and therapy to prevent escalating health, social and economic impacts. The spike protein (S) of this virus enables binding to the human receptor ACE2, and hence presents a prime target for vaccines preventing viral entry into host cells 1 . The S proteins from SARS-CoV-1 and SARS-CoV-2 are similar 2 , but structural differences in the receptor binding domain (RBD) preclude the use of SARS-CoV-1-specific neutralizing antibodies to inhibit SARS-CoV-2 3 . Here we used comparative pangenomic analysis of all sequenced Betacoronaviruses to reveal that, among all core gene clusters present in these viruses, the envelope protein E shows a variant shared by SARS and SARS-Cov2 with two completely-conserved key functional features, an ion-channel and a PDZ-binding Motif (PBM). These features trigger a cytokine storm that activates the inflammasome, leading to increased edema in lungs causing the acute respiratory distress syndrome (ARDS) 4-6 , the leading cause of death in SARS-CoV-1 and SARS-CoV-2 infection 7,8 . However, three drugs approved for human use may inhibit SARS-CoV-1 and SARS-CoV-2 Protein E, either acting upon the ion channel (Amantadine and Hexamethylene amiloride 9,10 ) or the PBM (SB203580 5 ), thereby potentially increasing the survival of the host, as already demonstrated for SARS-CoV-1in animal models. Hence, blocking the SARS protein E inhibits development of ARDS in vivo. Given that our results demonstrate that the protein E subcluster for the SARS clade is quasi-identical for the key functional regions of SARS-CoV-1 and SARS-CoV-2, we conclude that use of approved drugs shown to act as SARS E protein inhibitors can help prevent further casualties from COVID-2019 while vaccines and other preventive measures are being developed.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Early in planta detection of Xanthomonas axonopodis pv. punicae in pomegranate using 2 enhanced loop-mediated isothermal amplification assay 3 4 Abstract 33 Bacterial blight in pomegranate caused by Xanthomonas axonopodis pv. punicae (Xap) is an 34 increasing threat for pomegranate cultivation in India. To prevent the economic losses, it is pivotal 35 to detect the infection in latent stages rather than in later stages. We have developed an enhanced 36 method termed as loop-mediated isothermal amplification (LAMP) technique to evaluate for the 37 latent detection of Xap in pomegranate using six set of specific primers. Three DNA intercalating 38 dyes were used, such as Ethidium bromide, hydroxynaphthol blue (HNB) and SYBR Green which 39 resulted in visualising the positivity for LAMP assay. The reaction time and temperature were to 40 be 65°C from 30 min onwards, for the dyes and its sensitivity was observed up to 10 -7 ng in the 41 LAMP assay. For field applicability, LAMP assay detected Xap on 7 th day post infection while the 42 PCR amplified Xap after 11 th day post infection. Finally, the specificity of LAMP assay was 43 validated to be positive with ten Xap isolates for its accuracy and 29 non-Xap bacterial isolates 44 showed negative results. Moreover, this method could be used as a better alternative to PCR based 45 methods, for early detection of the pathogens. 46 47 50 Pomegranate (Punica granatum L.) is an important fruit crop widely cultivated from tropical to 51 temperate conditions [1]. India ranks first in the world for maximum area under pomegranate 52 cultivation (0.125 million hectares) and Maharashtra, Karnataka and Andhra Pradesh are the major 53 pomegranate growing states [2]. The bacterial blight caused by Xanthomonas axonopodis pv. 54 punicae (Xap) in pomegranate was first reported in India in 1952 and presently it has a serious 55 outbreak in all major pomegranate-growing states resulting in a major economic loss. Prominent 56 small water-soaked and dark irregular spots appearing on the leaf are the typical symptoms induced 57 by this pathogen. The pathogen infect the fruit, stem, branches causing cracking symptoms and 58 ultimately leads to yield loss up to 80 percent [3]. . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/328674 doi: bioRxiv preprint 59 Globally, it is estimated that a minimum 30-40% of the crop cultivated gets destroyed 60 annually due to pests and diseases [4]. In India, bacterial blight alone reduces the yield between 61 60 to 80 percent above global yield loss in pomegranate. To avoid yield loss, it is very important 62 and necessary to develop an efficiently sensitive pathogen detection method within incubation time 63 of the pathogen (i.e. time between the host infection and expression of the disease symptoms) 64 [5,6]. 65 The most common methods employed for detecting the plant pathogens include 66 immunological techniques like ELISA (enzyme-linked immunosorbent assay) [7] and molecular 67 methods such as PCR (polymerase chain reaction) [8] and Southern blotting [9]. The methods for 68 early detection of the plant pathogens in the field has to be designed in such a way that it has to be 69 competently very sensitive, specific, cost-effective and detectable with a minimal equipment 70 facilities [10]. For instance, the PCR based technique has its own drawbacks which require high 71 quality and quantity of DNA and amplification is complex that requires a thermal cycler for 72 changing the reaction temperature and requires relatively long time for the amplification process 73 to complete. Further, it also requires agarose gel electrophoresis and gel documentation to confirm 74 the amplification [11]. To overcome these limitations, researchers have shown interest in 75 developing an isothermal amplification technique termed as Loop-mediated isothermal 76 amplification (LAMP) [12]. LAMP is an auto-cycling strand displacement DNA synthesis 77 technique, which is carried out at a constant temperature and does not require a thermal-cycler. 78 The amplification is completed within 30-60 minutes at a constant temperature of 60-65º C, which 79 can be carried out in a simple water bath or heating block [13]. Moreover, LAMP is a cost effective 80 and it does not require a thermal cycler and electrophoretic unit, as the result can be inferred as 81 positive or negative with the colour change or turbidity observed by naked eye without spending 82 much time [14]. 83 The favorable target for bacterial pathogen diagnostic includes mainly, 16S ribosomal 84 RNA (rRNA) gene which is been widely used to identify the bacterial species among different 85 bacteria [15-17]. This diversity is unique for species and their sequences are presented inside their 86 nine (V1 -V9) hypervariable regions which constitutes useful targets [18]. These regions are 87 highly conserved stretches in bacteria, which facilitates the identification through PCR In principle, the LAMP assay recognizes six specifically designed primers that recognize 90 eight distinct sequences from the target bacterial DNA [20]. It consists of two outer primers (F3 91 and B3), two inner primers (FIP and BIP), a loop-forward (LF) and a loop-backward (LB). These 92 six primers ensure a high specificity for target DNA amplification. Moreover, these primers enable 93 generation of a stem-loop DNA for subsequent complex LAMP amplification, which includes self-94 priming reaction. A mixture of DNA, differing in the number of loops and length of the stem-loop 95 is produced as a final product [21]. By considering the advantages of LAMP assay, which is widely 96 used to detect broad range of microorganisms[22,23]. The present study was aimed to detect the 97 bacterial blight of pomegranate in early stages with minimum pathogen DNA template by 98 optimizing several DNA intercalating dyes with lower temperature in a shorter time with less 99 technical skills.100MATERIALS AND METHODSSource of bacterial blight material used for the study 102 Ten Xanthomonas axonopodis pv. punicae (Xap) isolates collected from four different regions of 103 Karnataka, India were used in this study (Fig 1) . Additionally, we have also isolated 29 non-104 Xanthomonas bacteria from pomegranate Table 1 . The pure culture Xap isolates were maintained 105 in a Nutrient Broth for 72 hours and the DNA was extracted using crude method. The quality and 106 quantity of the DNA were determined using Nanodrop (Model -DS-11 FX +, DeNovix, USA).\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "A novel coronavirus (SARS-CoV-2) has recently emerged as a global threat. As the epidemic progresses, many disease modelers have focused on estimating the basic reproductive number R 0 -the average number of secondary cases caused by a primary case in an otherwise susceptible population. The modeling approaches and resulting estimates of R 0 vary widely, despite relying on similar data sources. Here, we present a novel statistical framework for comparing and combining different estimates of R 0 across a wide range of models by decomposing the basic reproductive number into three key quantities: the exponential growth rate r, the mean generation intervalḠ, and the generation-interval dispersion κ. We then apply our framework to early estimates of R 0 for the SARS-CoV-2 outbreak. We show that many early R 0 estimates are overly confident. Our results emphasize the importance of propagating uncertainties in all components of R 0 , including the shape of the generation-interval distribution, in efforts to estimate R 0 at the outset of an epidemic.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Angiotensin-converting enzyme 2 (ACE2) has been suggested to be the cellular receptor for the new coronavirus (2019-nCoV) that is causing the coronavirus disease 2019 . Like other coronaviruses such as the SARS-CoV, the 2019-nCoV uses the receptor binding domain (RBD) of the surface spike glycoprotein (S protein) to engage ACE2. We most recently determined the structure of the full-length human ACE2 in complex with a neutral amino acid transporter B 0 AT1. Here we report the cryo-EM structure of the full-length human ACE2 bound to the RBD of the 2019-nCoV at an overall resolution of 2.9 Å in the presence of B 0 AT1. The local resolution at the ACE2-RBD interface is 3.5 Å, allowing analysis of the detailed interactions between the RBD and the receptor.Similar to that for the SARS-CoV, the RBD of the 2019-nCoV is recognized by the extracellular peptidase domain (PD) of ACE2 mainly through polar residues.Pairwise comparison reveals a number of variations that may determine the different affinities between ACE2 and the RBDs from these two related viruses. author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "To assess the viability of isolation and contact tracing to control onwards transmission from imported cases of 2019-nCoV.We developed a stochastic transmission model, parameterised to the 2019-nCoV outbreak. We used the model to quantify the potential effectiveness of contact tracing and isolation of cases at controlling a 2019 nCoV-like pathogen. We considered scenarios that varied in: the number of initial cases; the basic reproduction number R0; the delay from symptom onset to isolation; the probability contacts were traced; the proportion of transmission that occurred before symptom onset, and the proportion of subclinical infections. We assumed isolation prevented all further transmission in the model. Outbreakswere deemed controlled if transmission ended within 12 weeks or before 5000 cases in total.We measured the success of controlling outbreaks using isolation and contact tracing, and quantified the weekly maximum number of cases traced to measure feasibility of public health effort.Findings: While simulated outbreaks starting with only 5 initial cases, R0 of 1.5 and little transmission before symptom onset could be controlled even with low contact tracing probability, the prospects of controlling an outbreak dramatically dropped with the number of initial cases, with higher R0, and with more transmission before symptom onset. Across different initial numbers of cases, the majority of scenarios with an R0 of 1.5 were controllable with under 50% of contacts successfully traced. For R0 of 2.5 and 3.5, more than 70% and 90% of contacts respectively had to be traced to control the majority of outbreaks.The delay between symptom onset and isolation played the largest role in determining whether an outbreak was controllable for lower values of R0. For higher values of R0 and a large initial number of cases, contact tracing and isolation was only potentially feasible when less than 1% of transmission occurred before symptom onset.Interpretation: We found that in most scenarios contact tracing and case isolation alone is unlikely to control a new outbreak of 2019-nCov within three months. The probability of control decreases with longer delays from symptom onset to isolation, fewer cases ascertained by contact tracing, and increasing transmission before symptoms. This model can be modified to reflect updated transmission characteristics and more specific definitions of outbreak control to assess the potential success of local response efforts.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Two months after it was firstly reported, the novel coronavirus disease COVID-19 has already spread worldwide. However, the vast majority of reported infections have occurred in China. To assess the effect of early travel restrictions adopted by the health authorities in China, we have implemented an epidemic metapopulation model that is fed with mobility data corresponding to 2019 and 2020. This allows to compare two radically different scenarios, one with no travel restrictions and another in which mobility is reduced by a travel ban. Our findings indicate that i) travel restrictions are an effective measure in the short term, however, ii) they are ineffective when it comes to completely eliminate the disease. The latter is due to the impossibility of removing the risk of seeding the disease to other regions. Our study also highlights the importance of developing more realistic models of behavioral changes when a disease outbreak is unfolding.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Coronaviruses express a multifunctional papain-like protease, termed PLP2. PLP2 acts as 17 a protease that cleaves the viral replicase polyprotein, and a deubiquitinating (DUB) enzyme which 18 removes ubiquitin moieties from ubiquitin-conjugated proteins. Previous in vitro studies 19 implicated PLP2 DUB activity as a negative regulator of the host interferon (IFN) response, but 20 the role of DUB activity during virus infection was unknown. Here, we used X-ray structure-21 guided mutagenesis and functional studies to identify amino acid substitutions within the 22 ubiquitin-binding surface of PLP2 that reduced DUB activity without affecting polyprotein 23 processing activity. We engineered a DUB mutation (Asp1772 to Ala) into a murine coronavirus 24 and evaluated the replication and pathogenesis of the DUB mutant virus (DUBmut) in cultured 25 macrophages and in mice. We found that the DUBmut virus replicates similarly as the wild-type 26 virus in cultured cells, but the DUBmut virus activates an IFN response at earlier times compared 27 to the wild-type virus infection in macrophages, consistent with DUB activity negatively 28 regulating the IFN response. We compared the pathogenesis of the DUBmut virus to the wild-type 29 virus and found that the DUBmut-infected mice had a statistically significant reduction (p<0.05) 30 in viral titer in livers and spleens at day 5 post-infection, albeit both wild-type and DUBmut virus 31 infections resulted in similar liver pathology. Overall, this study demonstrates that structure-guided 32 mutagenesis aids the identification of critical determinants of PLP2-ubiquitin complex, and that 33 PLP2 DUB activity plays a role as an interferon antagonist in coronavirus pathogenesis.34ImportanceCoronaviruses employ a genetic economy by encoding multifunctional proteins that 36 function in viral replication and also modify the host environment to disarm the innate immune 37 response. The coronavirus papain-like protease 2 (PLP2) domain possesses protease activity, 38 All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/782409 doi: bioRxiv preprint which cleaves the viral replicase polyprotein, and also DUB activity (de-conjugating 39 ubiquitin/ubiquitin-like molecules from modified substrates) using identical catalytic residues. To 40 separate the DUB activity from the protease activity, we employed a structure-guided mutagenesis 41 approach and identified residues that are important for ubiquitin-binding. We found that mutating 42 the ubiquitin-binding residues results in a PLP2 that has reduced DUB activity but retains protease 43 activity. We engineered a recombinant murine coronavirus to express the DUB mutant and showed 44 that the DUB mutant virus activated an earlier type I interferon response in macrophages and 45 exhibited reduced pathogenesis in mice. The results of this study demonstrate that PLP2/DUB is 46 an interferon antagonist and a virulence trait of coronaviruses. 47 48 Coronaviruses (CoVs) are members of the order Nidovirales, which includes enveloped viruses 49 with large (~30kb), positive-sense single-stranded RNA genomes that yield a characteristic nested 50 set of subgenomic mRNAs during replication in the cytoplasm of infected cells (1, 2). The genome 51organization for coronaviruses is highly conserved, with the 5'-most two-thirds of the genome 52 encoding the replicase polyprotein, followed by sequences encoding the canonical structural 53 proteins: spike, envelope, membrane, and nucleocapsid. Many CoVs contain accessory genes, 54 which are interspersed among the genes for the structural proteins. Although these accessory genes 55 are not necessarily required for virus replication and are, in general, not highly conserved within 56 the virus family, many encode proteins that regulate the host response (3). Interestingly, 57 coronavirus replicase proteins, which are highly conserved, can also act as antagonists to block or 58 delay the host innate immune response to infection (1, 4-8). That a slew of coronavirus-encoded 59 accessory and non-accessory proteins have been shown to shape the host antiviral response 60 suggests that viral-mediated subversion of host defenses is an important element of infection. Here, 61 All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/782409 doi: bioRxiv preprint we focus on the viral protease/deubiquitinase (DUB) with the goal of assessing the role of DUB 62 activity in shaping the pathogenesis of mouse hepatitis virus (MHV), a model murine coronavirus.Coronavirus proteases are essential for viral replication as they are responsible for processing 64 the replicase polyprotein (2,(9)(10)(11). Murine coronavirus MHV encodes three proteases: two 65 papain-like proteases (PLP1 and PLP2) and one chymotrypsin-like protease (3CLpro, also termed 66 Mpro). MHV PLP2 is similar to the single papain-like protease (termed PLpro) of Severe Acute 67 Respiratory Syndrome Coronavirus (SARS-CoV) and Middle East Respiratory Syndrome 68 Coronavirus (MERS-CoV) (9, 12-16). We and others have revealed that PLP2 or PLpro of 69 multiple coronaviruses, including MHV, are multifunctional, not only capable of cleaving the viral 70 polyprotein but also possessing deubiquitinase (DUB) and deISGylating (de-conjugating ISG15 71 molecule from modified substrates) activities (9, 12, 13, 15-27). However, it has been challenging 72 to study the effects of a mutated DUB on virus replication and pathogenesis because the protease 73 and DUB activities share the same catalytic site, disruption of which is lethal for virus replication.74Therefore, it was necessary for us to identify residue(s) within the MHV protease/DUB-ubiquitin 75 binding surface that can be mutated to result in reduced DUB activity without affecting polyprotein 76 processing. In this report, we describe the X-ray structure-guided identification of such residues 77 and evaluate the replication, IFN antagonistic effect, and pathogenesis of a recombinant DUB 78 mutant MHV.79ResultsStructure-guided identification of PLP2 residues interacting with ubiquitin. To investigate 81 the role of viral deubiquitinating activity in coronavirus replication and pathogenesis, we first 82 needed to identify amino acid residues that, when mutated, result in reduced DUB activity while 83 All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/782409 doi: bioRxiv preprint 5 preserving the enzyme's deISGylating and protease activities, the latter being necessary for viral 84 replication. X-ray structural studies of SARS-and MERS-CoV papain-like protease/DUBs co-85 crystalized with ubiquitin modified at the c-terminus with a covalent warhead (aldehyde, 3-bromo- 86 propylamine, or propargylamine) allowed for identification of residues that are important for direct 87 interaction with ubiquitin (17, 18, 20). Here, we took a slightly different approach and mutated the 88 catalytic cysteine (C1716) of MHV PLP2 to a serine residue and co-crystallized it with free 89 ubiquitin. The X-ray structure of MHV PLP2 (C1716S) in complex with ubiquitin was determined 90 to a resolution of 1.85 Å and an overall Rfree = 19.6% and Rwork = 15.8% (PDB: 5WFI). The overall 91 structure of the MHV PLP2-ubiquitin complex is similar to other PLP2/PLpro ubiquitin-bound 92 structures (Fig. 1A). Ubiquitin binds within the palm region and is gripped by the zinc-fingers 93 motif while the C-terminus extends into the active site. 94 Next, we aligned the primary amino acid sequence of the MHV PLP2 domain with the 95 SARS-CoV and MERS-CoV papain-like protease domains (Fig. 1E) . The sequence alignment and 96 X-ray structure of the MHV PLP2-ubiquitin complex were then analyzed in conjunction with the 97 previous structural and mutagenesis studies on SARS-and MERS-CoV to identify candidate 98 residues that could be mutated to render a loss of DUB activity in vitro. From this analysis, we 99 identified three residues (R1803, D1772, F1812) in MHV PLP2 that form direct interactions with 100 ubiquitin (Fig. 1B-D) . Two of the side chain guanidinium nitrogens of R1803 form direct hydrogen 101 bonds with the backbone carbonyl oxygen of A46 in ubiquitin (Fig. 1B) . The two side chain 102 carboxylate oxygens of D1772 in MHV PLP2 interact with ubiquitin by forming direct bonds with 103 each of the guanidinium nitrogens of R42 and with one of the guanidinium nitrogens of R72 (Fig.   104 1C). Finally, F1812 forms Van der Waals contacts with the side chains of I44 and V70 and the 105 delta-carbon of R42 of ubiquitin (Fig. 1D) .106 All rights reserved. No reuse allowed without permission.To identify a mutant MHV PLP2 enzyme that 107 retains protease activity but exhibits reduced DUB and/or deISGylating activity, we performed 108 site-directed mutagenesis on each of the residues (R1803, D1772, F1812) by changing each to an 109 alanine to disrupt interactions with ubiquitin. Each mutant enzyme was expressed, purified, and 110 tested for its ability to hydrolyze three substrates: z-RLRGG-AMC, Ub-AMC, and ISG15-AMC.The activity of each mutant enzyme toward each substrate relative to the wild-type enzyme is 112 shown in Fig. 2A . All three mutants retained their ability to hydrolyze the peptide substrate but 113 each mutant had altered specificity toward Ub-AMC and ISG15-AMC substrates. Mutation of 114 F1812 resulted in a substantial decrease in hydrolysis of both Ub-AMC and ISG15-AMC (Class 115 I), whereas mutation of R1803 resulted in loss of activity only toward ISG15-AMC (Class II), and 116 mutation of D1772 resulted in loss of activity only toward Ub-AMC (Class III). 117 151 All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/782409 doi: bioRxiv preprint glycerophosphate, 1mM sodium orthovanadate, 1 µg/mL leupeptin, and 1mM 334 phenylmethylsulfonyl fluoride). Whole cell lysates were separated by SDS-PAGE and transferred 335 to PVDF membrane in transfer buffer (0.025M Tris, 0.192M glycine, 20% methanol) for 1 hour 336at 60 Volts at 4°C. Following this, the membrane was blocked using 5% dried skim milk in TBST 337 buffer (0.9% NaCl, 10mM Tris-HCl, pH7.5, 0.1% Tween 20) overnight at 4°C. The membrane 338 was incubated with either polyclonal rabbit anti-GFP antibody (A11122, Life Technologies) for 339 the protease assay, or mouse anti-flag (F3165, Sigma) for the DUB assay. The membrane was then 340 washed three times for 15 minutes in TBST buffer followed by incubation with either secondary 341 donkey anti-rabbit-HRP antibody (711-035-152, Jackson ImmunoResearch) or goat anti-mouse-342 HRP antibody (1010-05, SouthernBiotech). Then the membrane was washed three more times for 343 15 minutes in TBST buffer. Detection was performed using Western Lighting Chemiluminescence 344 Reagent Plus (PerkinElmer) and visualized using a FluoroChemE Imager (Protein Simple). The 345 expression of PLP2, β-actin, and calnexin were probed with mouse anti-V5 antibody (R960, 346 ThermoFisher), mouse anti-β-actin (A00702, Genscript), or mouse anti-calnexin antibody (2433S, 347 Cell Signaling), respectively. 348 Biosensor live cell assay 349The protease activity of PLP2 was also assessed using a biosensor live cell assay as 350 described previously (14). Briefly, HEK293T cells in a 96 black-wall plate were transfected with 351 37.5 ng pGlo-RLKGG construct and 50 ng PLP2 expression plasmids. GloSensor (E1290, 352 Promega) reagent diluted in DMEM+10% FCS was added at 18 hpi. Plates were read using a 353 luminometer (Veritas) every hour over a course of 5 hours. 354 All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "In March 2020, Coronavirus Disease 2019 originating from Wuhan, China, has become pandemic. Based on epidemic data from China we constructed and constrained a simple Susceptible-Infected-Infectious-Excluded (SIIE) model. The model is characterized by only three parameters: average incubation period, contact rate r C , and exclusion rate r E . These two rates control the daily multiplication coefficient β describing epidemic growth in its early exponential phase, during which the number of excluded (confirmed) individuals grows as E = E0 exp(α t) with α = ln β. The rates can be modified by non-therapeutic interventions: r C can be reduced by quarantine, while r E can be increased by efficient testing, enabling isolation of infectious individuals. In order to contain exponential epidemic growth, quotient r C /r E has to be reduced to 1. Based on the model, we estimate that the quarantine imposed in China on January 23, 2020 resulted in about 50-fold reduction of this quotient, allowing to terminate the exponential growth phase and enter the phase of exponential regression of daily new cases. In contrast, quarantine imposed in northern, most affected, Italian provinces on February 21 resulted in about 3-fold reduction of r C /r E and corresponding decrease of β from about 1.5 to about 1.2; further 5-fold reduction is necessary to terminate exponential growth. Even higher reduction should take place in France, Germany and Spain for which β > 1.3 (as of March 10, 2020). Considering longer-term dynamics in the case when the initial exponential growth is not contained by quarantine, we analyzed a scenario in which r E decreases in response to the surge of daily new cases which convinces people to isolate themselves. In such case, initial abrupt epidemic growth is followed by a slow decrease over years. This scenario, although economically and socially devastating, grants time to develop, produce, and distribute vaccine.. CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.is the (which was not peer-reviewed) The copyright holder for this preprint .\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "In this paper, we estimate the reproductive number R 0 of COVID-19 based on Wallinga and Lipsitch framework [11] and a novel statistical time delay dynamic system. We use the observed data reported in CCDC's paper to estimate distribution of the generation interval of the infection and apply the simulation results from the time delay dynamic system as well as released data from CCDC to fit the growth rate. The conclusion is: Based our Fudan-CCDC model, the growth rate r of COVID-19 is almost in [0.30, 0.32] which is larger than the growth rate 0.1 estimated by CCDC [9], and the reproductive number R 0 of COVID-19 is estimated by 3.25 ≤ R 0 ≤ 3.4 if we simply use R = 1 + r * T c with T c = 7.5, which is bigger than that of SARS. Some evolutions and predictions are listed.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "With an ongoing threat posed by circulating zoonotic strains, new strategies are required to 8 prepare for the next emergent coronavirus (CoV). Previously, groups had targeted conserved 9 coronavirus proteins as a strategy to generate live-attenuated vaccine strains against current 10 and future CoVs. With this in mind, we explored whether manipulation of CoV NSP16, a 11 conserved 2'O methyltransferase (MTase), could provide a broad attenuation platform against 12 future emergent strains. Using the SARS-CoV mouse model, a NSP16 mutant vaccine was 13 evaluated for protection from heterologous challenge, efficacy in the aging host, and potential 14 for reversion to pathogenesis. Despite some success, concerns for virulence in the aged and 15 potential for reversion makes targeting NSP16 alone an untenable approach. However, 16 combining a 2'O MTase mutation with a previously described CoV fidelity mutant produced a 17 vaccine strain capable of protection from heterologous virus challenge, efficacy in aged mice, 18 and no evidence for reversion. Together, the results indicate that targeting the CoV 2'O MTase 19 in parallel with other conserved attenuating mutations may provide a platform strategy for 20 rapidly generating live-attenuated coronavirus vaccines.. CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309591 doi: bioRxiv preprint 31 attenuation as a major component of future live-attenuated coronavirus vaccines.. CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309591 doi: bioRxiv preprint 77 However, the broad conservation of NSP16 coupled with robust viral yields permitted its use in 78 combination with another attenuating mutation in NSP14, exonuclease (ExoN) activity.Targeting both conserved coronavirus activities produced a stable, attenuated virus capable of 80 protection from heterologous challenge, efficacy in aged mice, and absence of reversion in 81 immunocompromised models. Together, the results indicate that targeting coronavirus NSP16 82 may be a critical component of a future live attenuated coronavirus vaccine approach. 83 84 . CC-BY-NC-ND 4.0 International license is made available under aThe copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309591 doi: bioRxiv preprint 107 SARS-CoV induce severe weight loss, fail to recover, and several succumb to infection (Fig.   108   1B) . Notably, while the SARS NSP16 mutant induces weight loss, mice recover from infection 109 diverging from wild-type infection 3 days post infection. Examining viral replication, the NSP16 110 . CC-BY-NC-ND 4.0 International license is made available under aThe copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309591 doi: bioRxiv preprint 130 (SHC014-MA15). Our results indicate that the attenuated 2'O MTase SARS-CoV mutant is 131 capable of inducing robust protection from the zoonotic group 2B spike virus in terms of 132 disease. Weight loss data, viral lung titer, and hemorrhage score indicated little to no disease in 133 mice vaccinated with dNSP16 compared to mock vaccinated mice ( Fig. 2A-C) . Importantly,while not equivalent to neutralization of wild-type SARS-CoV, sera from dNSP16-vaccinated 135 mice were able to effectively block the SHC014 chimeric virus, likely contributing to protection\n",
      "\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Infectious bronchitis (IB) causes significant economic losses in the global poultry industry. Control of infectious bronchitis is hindered by the genetic diversity of the causative agent, infectious bronchitis virus (IBV), which has led to the emergence of several serotypes that lack complete serologic cross-protection. While serotyping by definition requires immunologic characterization, genotyping is an efficient means to identify IBVs detected in samples. Sanger sequencing of the S1 subunit of the spike gene is currently used to genotype IBV; however, the universal S1 PCR was created to work from cultured IBV and it is inefficient at detecting mixed isolates. This paper describes a MinION-based AmpSeq method that genetically typed IBV from clinical samples, including samples with multiple isolates. Total RNA was extracted from fifteen tracheal scrapings and choanal cleft swab samples, randomly reverse transcribed, and PCR amplified using modified S1-targeted primers. Amplicons were barcoded to allow for pooling of samples, processed per manufacturer's instructions into a 1D MinION sequencing library, and sequenced on the MinION. The AmpSeq method detected IBV in 13 of 14 IBV-positive samples.AmpSeq accurately detected and genotyped both IBV lineages in three of five samples containing two IBV lineages. Additionally, one sample contained three IBV lineages, and AmpSeq accurately detected two of the three. Strain identification, including detection of different strains from the same lineage, was also possible with this AmpSeq method. The results demonstrate the feasibility of using MinION-based AmpSeq for rapid and accurate identification and lineage typing of IBV from oral swab samples.Infectious bronchitis virus is an enveloped, pleomorphic gammacoronavirus with an unsegmented, single-stranded, positive-sense, 26-27.8 Kb, RNA genome that encodes the nonstructural polyprotein,1a and 1b, and several structural proteins: spike (S), envelop (E), membrane (M) and nucleocapsid (N). 38,41 In addition, two accessory genes, expressing 3a, 3b and 5a and 5b, respectively, have also been described. 6, 14, 38 The S protein is highly glycosylated and post-translational cleavage leads to two subunits: S1 and S2. 10,48 Besides acting as the viral attachment protein, the S1 protein is a major target of neutralizing antibodies. 7 As with many attachment proteins that are targets of virus-neutralizing antibodies, the S1 subunit is highly diverse with almost 50% of the amino acids differing among IBV serotypes. 2,21,38 Such variation leads to important biological differences between IBV serotypes and the emergence of novel variants. More than 60 serotypes of IBV have been reported, but the most common serotypes of All rights reserved. No reuse allowed without permission.Massachusetts. 16 This genetic diversity leads to emergence of new serotypes and a lack of complete cross-serotype protection by vaccines. 29 The correlation of IBV genotypes and serotypes of IBV has been reported 45 ; therefore, accurate genotypic identification of IBV is an important step to diagnose IBV in clinical respiratory cases, ensure selection of proper IBV vaccines for use in vaccination programs, and to better understand the epidemiology of this global virus.One of the recent comprehensive classification schemes for IBV uses S1 gene sequencebased phylogeny of IBV and identified six genotypes (I-VI), 32 sub-genotypic lineages, and a number of inter-lineage recombinants in global strains of IBV. 44 Among the six genotypes, genotype I (G-I) is the most diverse group of viruses, with 27 unique lineages. 44 As such, sequencing of the S1 subunit provides important information regarding the classification of IBV within samples.The genetic classification of IBV has relied on genotype-specific RT-qPCR assays, serotype-specific S1 RT-qPCR 32,40 and/or pan-IBV S1 RT-PCR assays coupled with Sanger sequencing. 8,31,49 Genotype-specific RT-qPCRs are limited to short fragments, which may miss important changes in the S1 (~1.6 kb) 44 outside the short target. Additionally, since the target is short, the primers lie within the variable regions of the S1 and may require a different assay for each genotype. Lastly, the pan-IBV S1 primers 1, 17, 22, 23, 25, 33 , while only requiring one reaction, have relatively low sensitivity and are typically used only on egg-cultured virus, which adds an additional, time consuming step and many diagnostic labs do not have SPF embryos readily available. Lastly, cloning of PCR products has been used to detect multiple IBV sub-populations in the sample 30 ; however, this is inefficient when potentially dealing with multiple (three or four) All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/634600 doi: bioRxiv preprint IBV types. As such, the development of a pan-IBV sequencing method to rapidly determine the genotype(s) in samples would aid in lineage typing and tracking of IBV genotypes circulating in poultry flocks.Recently, third generation sequencing technology has been used for detection of viral nucleotides and sequencing ultra-long DNA molecules. 34 The MinION Nanopore sequencer, a new DNA sequencing technology that allows for rapid, in-house, real-time technique detection and differentiation of IBV lineages, may be cost effective and useful in the field. 19 Ampliconbased sequencing has also been used to amplify specific regions of Newcastle disease 3,13 , infectious laryngotracheitis virus 39 Zika 34 , Ebola 35 and avian influenza viruses 46 , by simple RT-PCR and then sequencing on the MinION device. Real-time data analysis, the lack of significant start-up cost investment or maintenance expenses, simultaneous and sequential multiplexing unique to MinION, and the ability to sequence long DNA molecules so that primers are in conserved regions while the product contains the variable region are the features that make this technology highly feasible to be used for disease diagnostics.Accurate identification of IBV genotypes from samples, including detection of multiple isolates from a single sample, is crucial to respiratory disease diagnosis, selection of appropriate IBV vaccines, and epidemiologic studies. Therefore, the aim of the current study is to create a single, amplicon-based protocol to sequence IBV-S1 gene and develop a sequence analysis workflow to identify IBV isolates from clinical swab samples. This method provides a useful assay for IBV and a model for the development of future amplicon sequencing (AmpSeq) based assays.All rights reserved. No reuse allowed without permission.A universal S1 primer set, 27 tailed with the MinION universal adapter sequence of 22 nucleotides (underlined, Table 2 ) to allow barcoding of amplicons, was used for targeted amplification of the IBV S1 gene for IBV. The PCR reaction mixture (Expand High Fidelity PCR system, Roche Diagnostics, Mannheim, Germany) was composed of 10 µl of cDNA, 1 µl of 10 µM forward primer, 1 µl of 10 µM reverse primer, 2.5 µl of 10× Expand high fidelity buffer, 1 µl of Expand High Fidelity Enzyme Mix, and 1 µl of 10mM dNTPs, to a final volume to 25 µl with 9 µl of nuclease-free water. The following thermocycling conditions were used for amplicon synthesis: denaturation at 95 °C for 2 min; 30 cycles of 94 °C for 1 min, 45 °C for 2 min, and 72 °C for 2 minutes, and final extension at 72 °C for 10 min. See Table 2 for details about all three primer sets used in this study. PCR products were electrophoresed in 1.5% agarose with SYBR Safe DNA gel stain (Invitrogen) for visual evaluation. Amplified DNA was purified by Agencourt AMPure XP beads (Beckman Coulter, USA) at 1.6:1 (volumetric bead:DNA) and quantified using the dsDNA High Sensitivity Assay Kit on a Qubit® Fluorimeter 3.0.The amplicons obtained from the tailed S1 primer set were then used to prepare MinIONcompatible DNA libraries. Briefly, each of the amplicons were diluted to 0.5 nM and amplified using barcoding primers (1D PCR Barcoding Amplicon Kit, Oxford Nanopore Technologies, [ONT], UK) and LongAmp Taq 2X Master Mix (New England Biolabs, MA, USA) with the following conditions: 95 °C for 3 min; 15 cycles of 95 °C for 15 seconds, 62 °C for 15 seconds, 65 °C for 80 seconds; and 65 °C for 80 seconds. The barcoded amplicons were bead purified (1:1.4; bead:solution), pooled (Run1; pool of 10 samples, Run2; pool of 5 samples) into a single All rights reserved. No reuse allowed without permission.Centrifuge 24 , Kraken 50 , QIIME 5 and Mothur 37 have been used to identify and profile microbial species; however, the high error rate 20 of Nanopore reads translates to poor classification accuracy for many of these tools. Alternatively, de novo 3,26 or reference-based 18 assembly methods have been used in MinION and other deep sequencing platforms. Using a strategy similar to other unbiased laboratory tests (e.g., standard bacterial cultures), an approach was developed to maximize usage of reads (i.e., reads are not discarded based on pre-set length or abundance requirements, similar to how a single colony may be interpreted as a significant result). This approach uses read-based classification against a database containing an equal number of representative sequences per lineage to detect and classify the reads based on their lineage, before conducting read-based classification to the isolate level by using all available IBV sequences in a separate database. Finally, lineage-clustered isolate-level read assignments are interpreted by a veterinary diagnostician to result in reads available for consensus building.The use of a final consensus alignment helps to overcome the individual error rate of MinION sequencing. 3 Similarly, the absence of predetermined metrics used in de novo assembly allows for the informed decision as to how many consensus sequences to build, a bioinformatics problem when dealing with clinical cases that can contain more than one isolate (e.g., similar to how there is not a predetermined number of significant bacterial colonies). Confirmatory followup tests (e.g., RT-PCR) may be needed when dealing with low number of reads in clinical samples; however, future testing of this new technology will allow for creating standards for such confirmatory testing.Taken together, these results suggest the application of MinION-based AmpSeq, specifically for detecting IBV isolates from clinical samples within a few days, compared to several days to weeks and multiple diagnostic assays to culture and detect multiple IBV All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Backgrounds: In December 2019, a novel coronavirus (COVID-19) pneumonia hit Wuhan, Hubei Province, China and spread to the rest of China and overseas. The emergence of this virus coincided with the Spring Festival Travel Rush in China. It is possible to estimate total number of cases of COVID-19 in Wuhan, by 23 January 2020, given the cases reported in other cities and population flow data between cities. Methods: We built a model to estimate the total number of cases in Wuhan by 23 January 2020, based on the number of cases detected outside Wuhan city in China, with the assumption that if the same screening effort used in other cities applied in Wuhan. We employed population flow data from different sources between Wuhan and other cities/regions by 23 January 2020. The number of total cases was determined by the maximum log likelihood estimation. Findings: From overall cities/regions data, we predicted 1326 (95% CI: 1177, 1484), 1151 (95% CI: 1018, 1292) and 5277 (95% CI: 4732, 5859) as total cases in Wuhan by 23 January 2020, based on different source of data from Changjiang Daily newspaper, Tencent, and Baidu. From separate cities/regions data, we estimated 1059 (95% CI: 918, 1209), 5214 (95% CI: 4659, 5808) as total cases in Wuhan in Wuhan by 23 January 2020, based on different sources of population flow data from Tencent and Baidu. Conclusion: Sources of population follow data and methods impact the estimates of local cases in Wuhan before city lock down.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Domain growth plays an important role in many biological systems, and so the inclusion of domain growth in models of these biological systems is important to understanding how these biological systems function. In this work we present methods to include the effects of domain growth on the evolution of spatial correlations in a continuum approximation of a lattice-based model of cell motility and proliferation. We show that, depending on the way in which domain growth is implemented, different steady-state densities are predicted for an agent population. Furthermore, we demonstrate that the way in which domain growth is implemented can result in the evolution of the agent density depending on the size of the domain. Continuum approximations that ignore spatial correlations cannot capture these behaviours, while those that account for spatial correlations do. These results will be of interest to researchers in developmental biology, as they suggest that the nature of domain growth can determine the characteristics of cell populations. * ross@maths.ox.ac.ukIn this work we examine how domain growth affects the evolution of spatial correlations between agents in IBMs, and importantly, how the way in which domain growth is implemented affects spatial correlations differently. To do so we use an agent-based (our agents represent 2 author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/041509 doi: bioRxiv preprint N x (t) by N y (t), where N x (t) is the number of lattice sites in a row and N y (t) is the number of sites in a column. All simulations are performed with periodic boundary conditions, and throughout this work the lattice spacing, ∆, is equal to one. For notational convenience we shall now write N x (t) as N x , and N y (t) as N y .In the IBM each agent is initially assigned to a lattice site, from which it can move or proliferate into an adjacent site. If an agent attempts to move into a site that is already occupied, 3 author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Estimation of the fraction and contagiousness of undocumented novel coronavirus infections is critical for understanding the overall prevalence and pandemic potential of this disease. Many mild infections are typically not reported and, depending on their contagiousness, may support stealth transmission and the spread of documented infection.Here we use observations of reported infection and spread within China in conjunction with mobility data, a networked dynamic metapopulation model and Bayesian inference, to infer critical epidemiological characteristics associated with the emerging coronavirus, including the fraction of undocumented infections and their contagiousness.We estimate 86% of all infections were undocumented (95% CI: [82%-90%]) prior to the Wuhan travel shutdown (January 23, 2020). Per person, these undocumented . CC-BY 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.(which was not peer-reviewed) The copyright holder for this preprint .\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Development of antiviral therapeutics emphasizes minimization of the effective dose and maximization 16 of the toxic dose, first in cell culture and later in animal models. Long-term success of an antiviral 17 therapeutic is determined not only by its efficacy but also by the duration of time required for drug-18 resistance to evolve. We have developed a microfluidic device comprised of ~6000 wells, with each 19 well containing a microstructure to capture single cells. We have used this device to characterize 20 enterovirus inhibitors with distinct mechanisms of action. In contrast to population methods, single-cell 21 analysis reveals that each class of inhibitor interferes with the viral infection cycle in a manner that can 22\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Honey bees are eusocial insects, living in a colony that is often described as a superorganism. RNA mobility among cells of an organism has been documented in plants and animals. Here we show that RNA spreads further in honey bees, and is horizontally transferred between individuals and across generations. We found that honey bees share biologically active RNA through secretion and ingestion of worker and royal jellies. Such RNA initiates RNA interference, which is a known defense mechanism against viral infection. Furthermore, we characterized diverse RNA profiles of worker and royal jelly, including fragmented viral RNA. Our findings demonstrate a transmissible RNA pathway with potential roles in social immunity and epigenetic signaling among members of the hive.One of the characteristics of RNA interference (RNAi) is systemic spread of the silencing signal among cells and tissues throughout the organism. Systemic RNAi, initiated by double-stranded RNA (dsRNA) ingestion, has been reported in diverse invertebrates, including honey bees, demonstrating environmental RNA uptake that undermines homologous gene expression. However, the question why any organism would take up RNA from the environment has remained largely unanswered. Here, we report on horizontal RNA flow among honey bees mediated by secretion and ingestion of worker and royal jelly diets. We show that ingested dsRNA spreads through the bee's hemolymph associated with a protein complex. The systemic dsRNA is secreted with the jelly and delivered to larvae via ingestion. Furthermore, we demonstrate that transmission of jelly-secreted dsRNA to larvae is biologically active and triggers gene knockdown that lasts into adulthood. Finally, RNA extracted from worker and royal jellies harbor differential naturally occurring RNA populations. Some of these RNAs corresponded to honey bee protein coding genes, transposable elements, non-coding RNA as well as bacteria, fungi and viruses. These results reveal an inherent property of honey bees to share RNA among individuals and generations. Thus, our findings suggest a transmissible RNA pathway, playing a role in social immunity and epigenetic signaling between honey bees and potentially among other closely interacting organisms.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Coronaviruses (CoV) are positive-sense RNA viruses that infect numerous mammalian and avian 23 species and are capable of causing severe and lethal disease in humans. CoVs encode several 24 innate immune antagonists that interact with the host innate immune response to facilitate 25 efficient viral replication. CoV non-structural protein 14 (nsp14) encodes 3'-to-5' 26 exoribonuclease activity (ExoN), which performs a proofreading function and is required for 27 high-fidelity replication. Outside of the order Nidovirales, arenaviruses are the only RNA viruses 28 that encode an ExoN, which functions to degrade dsRNA replication intermediates. In this study, 29we tested the hypothesis that CoV ExoN may also function to antagonize the innate immune 30 response. We demonstrate that viruses lacking ExoN activity [ExoN(-)] are sensitive to cellular 31 pretreatment with interferon beta (IFN-β) in a dose-dependent manner. In addition, ExoN(-) virus 32 replication was attenuated in wild-type bone marrow-derived macrophages (BMMs) and partially 33 restored in interferon alpha/beta receptor deficient (IFNAR-/-) BMMs. ExoN(-) virus replication 34 did not result in IFN-β gene expression, and in the presence of an IFN-β-mediated antiviral state, 35ExoN(-) viral RNA levels were not substantially reduced relative to untreated. However, ExoN(-36 ) virus generated from IFN-β pretreated cells had reduced specific infectivity and decreased 37 relative fitness, suggesting that ExoN(-) virus generated during an antiviral state is less viable to 38 establish a subsequent infection. Overall, our data suggest MHV ExoN activity is required for 39 resistance to the innate immune response and antiviral mechanisms affecting the viral RNA 40 sequence and/or an RNA modification act on viruses lacking ExoN activity. 41 42 43 All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The ongoing outbreak of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) started in the end of 2019 in China has triggered a global public health crisis. Previous studies have shown that SARS-CoV-2 infects cells by binding angiotensin-converting enzyme 2 (ACE2), which is the same as SARS-CoV. The expression and distribution of ACE2 in the pancreas are unknown. At the same time, the injury of pancreas after SARS-CoV-2 infection has not been concerned. Here, we collected public datasets (bulk RNA-seq and single-cell RNA-seq) to indicate the expression and the distribution of ACE2 in pancreas (in both exocrine glands and islets). And further, clinical data including mild and severe patients with COVID-19 demonstrated there existed mild pancreatitis. In the 67 severe cases, 11 patients (16.41%) showed elevated levels of both amylase and lipase, and 5 patients (7.46%) showed imaging alterations. Only one patient (1.85%) showed elevated levels of both amylase and lipase in 54 mild cases, without imaging changes. Our study revealed the phenomenon and possible cause of mild pancreatic injury in patients with COVID-19. This suggests that pancreatitis after SARS-CoV-2 infection should also be paid attention in clinical work.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Viruses interact with hundreds to thousands of proteins in mammals, yet adaptation 6 against viruses has only been studied in a few proteins specialized in antiviral defense.Whether adaptation to viruses typically involves only specialized antiviral proteins or 8 affects a broad array of proteins is unknown. Here, we analyze adaptation in ~1,300 9 virus-interacting proteins manually curated from a set of 9,900 proteins conserved 10 across mammals. We show that viruses (i) use the more evolutionarily constrained 11 proteins from the cellular functions they hijack and that (ii) despite this high constraint, 12 virus-interacting proteins account for a high proportion of all protein adaptation in 13 humans and other mammals. Adaptation is elevated in virus-interacting proteins across 14 all functional categories, including both immune and non-immune functions. Our results 15 demonstrate that viruses are one of the most dominant drivers of evolutionary change 16 across mammalian and human proteomes.17\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Cases with coronavirus disease 2019 (COVID-19) emigrated from Wuhan escalated the risk of spreading in other cities. This report focused on the outside-Wuhan patients to assess the transmission and clinical characteristics of this illness.Contact investigation was conducted on each patient who admitted to the assigned hospitals in Hunan Province (geographically adjacent to Wuhan) from Jan 22, 2020 to Feb 12, 2020.Demographic, clinical, laboratory and radiological characteristics, medication therapy and outcomes were collected and analyzed. Patients were confirmed by PCR test.Of the 104 patients, 48 (46.15%) were imported cases and 56 (53.85%) were indigenous cases; 93 (89.42%) had a definite contact history with infections. Family clusters were the major body of patients. Transmission along the chain of 3 \"generations\" was observed. Mean age was 43 (rang, 8-84) years (including 3 children) and 49 (47.12%) were male. Most patients had typical symptoms, 5 asymptomatic infections were found and 2 of them infected their relatives. The median incubation period was 6 (rang, 1-32) days, of 8 patients ranged from 18 to 32 days. Just 9 of 16 severe patients required ICU care. Until Feb 12, 2020, 40 (38.46%) discharged and 1 (0.96%) died. For the antiviral treatment, 80 (76.92%) patients received traditional Chinese medicine therapy. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Dengue virus (DENV) represents a public health and economic burden in affected countries. The availability of genomic data is key to understand viral evolution and dynamics, supporting improved control strategies. Currently, the use of High Throughput Sequencing (HTS) technologies, which can be applied both directly to patient samples (shotgun metagenomics) and to PCR amplified viral sequences (targeted metagenomics), is the most informative approach to monitor the viral dissemination and genetic diversity.Despite many advantages, these technologies require bioinformatics expertise and appropriate infrastructure for the analysis and interpretation of the resulting data. In addition, the many software solutions available can hamper reproducibility and comparison of results.Here we present DEN-IM, a one-stop, user-friendly, containerised and reproducible workflow for the analysis of DENV sequencing data, both from shotgun and targeted metagenomics approaches. It is able to infer DENV coding sequence (CDS), identify serotype and genotype, and generate a phylogenetic tree. It can easily be run on any UNIX-like system, from local machines to high-performance computing clusters, performing a comprehensive analysis without the requirement of extensive bioinformatics expertise.Using DEN-IM, we successfully analysed two DENV datasets. The first comprised 25 shotgun metagenomic sequencing samples of varying serotype and genotype, including a spiked sample containing the existing four serotypes. The second dataset consisted of 106 targeted metagenomics samples of DENV 3 genotype III where DEN-IM allowed detection of the intra-genotype diversity.The DEN-IM workflow, parameters and execution configuration files, and documentation are freely available at https://github.com/B-UMMI/DEN-IM. The Dengue virus (DENV), a single-stranded positive-sense RNA virus belonging to the Flavivirus 2 genus, is one of the most prevalent arboviruses and is mainly concentrated in tropical and subtropical 3 regions. Infection with DENV results in symptoms ranging from mild fever to haemorrhagic fever and 4 shock syndrome [1]. Transmission to humans occurs through the bite of Aedes mosquitoes namely 5 Aedes aegypti and Aedes albopictus [2] In 2010, it was predicted that the burden of dengue disease 6 reached 390 million cases per year worldwide [3]. The high morbidity and mortality of dengue makes 7 it the arbovirus with the highest clinical significance [4]. 8 The viral genome of ∼11,000 nucleotides, consists of a Coding Sequence (CDS) of approximately 910.2 Kb that is translated into a single polyprotein encoding three structural proteins (capsid -C, 10 premembrane -prM, envelope -E) and seven non-structural proteins (NS1, NS2A, NS2B, NS3, NS4A, 11 NS4B and NS5. Additionally, the genome contains two Non-Coding Regions (NCRs) at their 5' and 12 3' ends [5].DENV can be classified into four serotypes (1, 2, 3 and 4), differing from each other by 25% to 14 40% at the amino acid level. They are further classified into genotypes that vary by up to ∼3% 15 at the amino acid level [2]. The DENV-1 serotype comprises five genotypes (I-V), DENV-2 groups 16 six (I-VI, also named American, Cosmopolitan, Asian-American, Asian II, Asian I and Sylvatic), 17 DENV-3 four (I-III and V), and DENV-4 also four (I-IV). 18 DENV is a significant public health challenge in countries where the infection is endemic due 19 to the high health and economic burden. Despite the emergence of novel therapies and ecological 20 strategies to control the mosquito vector, there are still important knowledge gaps in the virus biology 21 and its epidemiology [2].The implementation of a surveillance system relying on High Throughput Sequencing (HTS) 23 technologies allows the simultaneous identification and surveillance of DENV cases. Due to the 24 high sensitivity of these technologies, previous studies showed that viral sequences can be directly 25 obtained from patient sera using a shotgun metagenomics approach [6]. Alternatively, HTS can be 26 used in a targeted metagenomics approach in which a PCR step is used to pre-amplify viral sequences 27 before sequencing. In recent years, HTS has been successfully used as a tool for identification of 28 DENV directly from clinical samples [6, 7]. This also allows the rapid identification of the serotype 29 and genotype important for disease management as the genotype may be associated with disease 30 outcome [8].31 Several initiatives aim to facilitate the identification of the DENV serotype and genotype from 32 HTS data. The Genome Detective project (https://www.genomedetective.com/) offers an online 33 Dengue Typing Tool (https://www.genomedetective.com/app/typingtool/dengue/) relying on 34 BLAST and phylogenetic methods in order to identify the closest serotype and genotype, but it 35 requires as input assembled genomes in FASTA format. Alternatively, the same project offers 36 a Genome Detective Typing Tool (https://www.genomedetective.com/app/typingtool/virus/) 37 identifying viruses present in a sample.We developed DEN-IM as a ready-to-use, one-stop, reproducible bioinformatic analysis workflow 39 for the processing and phylogenetic analysis of DENV using paired-end raw HTS data. DEN-IM is 40 implemented in Nextflow [9], a workflow manager software that uses Docker (https://www.docker. 41 com) containers with pre-installed software for all the workflow tools. The DEN-IM workflow, as well 42 as parameters and documentation, are available at https://github.com/B-UMMI/DEN-IM. 43 3 The DEN-IM Workflow 44 DEN-IM is a user-friendly automated workflow allowing the analysis of shotgun or targeted metage-45 nomics data for the identification, serotyping, genotyping, and phylogenetic analysis of DENV. It is 46 implemented in Nextflow, a workflow management system that allows the effortless deployment and 47 3 . CC-BY 4.0 International license author/funder. It is made available under aThe copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/628073 doi: bioRxiv preprint execution of complex distributed computational workflows.The workflow is composed of five blocks: 1. Quality Control and Trimming, 2. Retrieval of DENV 49 sequences, 3. Assembly, 4. in silico Typing, and 5. Phylogeny, described in more detail in Figure 1 . 50 DEN-IM accepts as input raw paired-end sequencing data (FASTQ files), and informs the user with 51 an interactive HTML report with information on the quality control, mapping, assembly typing and 52 phylogenetic analysis, as well as all the output files of the whole pipeline. 53 3.1 1. Quality Control and Trimming 54 The Quality Control (QC) and Trimming block starts with a process to verify the integrity of the 55 input data. If the sequencing files are corrupted, the execution of the analysis of that sample is 56 terminated. 57 The sequences are then processed by FastQC (https://www.bioinformatics.babraham.ac.uk/ 58 projects/fastqc/, version 0.11.7) to determine the quality of the individual base pairs of the 59 raw data files. The low quality bases and adapter sequences are trimmed by Trimmomatic [10] 60 (version 0.36). By default, Trimmomatic removes the Illumina Nextera, TruSeq2 and TruSeq3 adapter 61 sequences that are detected, and relies on Phred scores [11] and FastQC analysis data to determine 62 to which extend the 5' and 3' ends of the reads need to be trimmed. The crop and headcrop setting 63 is calculated per sample, based on the GC ratio of each half of the read. By default, the window size 64 is set to 5 nucleotides with a minimum average quality of 20. In addition, paired-end reads with a 65 read length shorter than 55 nucleotides after trimming are removed from further analyses. 66 Lastly, the low complexity sequences, containing over 50% of poly-A, poly-N or poly-T nucleotides, 67 are filtered out of the raw data using PrinSeq [12] (version 0.10.4).68 3.2 2. Retrieval of DENV Sequences 69 In the second step, DENV sequences are selected from the sample using Bowtie2 [13] (version 70 2.2.9) and Samtools [14] (version 1.4.1). As a reference we provide the DENV mapping database, a 71 curated DENV database composed of 3830 complete DENV genomes (see Methods, DENV Reference 72Database). A permissive approach is followed by allowing for mates to be kept in the sample even 73 when only one read maps to the database in order to keep as many DENV derived reads as possible.The output of this step is a set of processed reads of putative DENV origin. 75 3.3 3. Assembly 76 DEN-IM applies a two assembler approach to generate assemblies of the DENV CDS. To obtain a 77 high confidence assembly, the processed reads are first de novo assembled with SPAdes [15] (version 78 3.12.0). If the full CDS fails to be assembled into a single contig, the data is re-assembled with 79 MEGAHIT assembler [16] (version 1.1.3), a more permissive assembler developed to retrieve longer 80 sequences from metagenomics data. The resulting assemblies are corrected with Pilon [17] (version 81 1.22) after mapping the processed reads to the assemblies with Bowtie2 [13]. 82 If more than one complete CDS is present in a sample, each of the sequences will follow the rest 83 of the DEN-IM workflow independently. If no full CDS is assembled neither with SPAdes nor with 84 MEGAHIT, the processed reads are passed on to the next step for consensus generation by mapping, 85 effectively constituting DEN-IM's two pronged approach using both assemblers and mapping.86 3.4 4. Typing 87 For each DENV complete CDS, the serotype and genotype is determined with the Seq Typing tool 88 (https://github.com/B-UMMI/seq_typing, version 2.0) [18] using BLAST [19] and the custom 89 Typing database of DENV containing 161 complete sequences (see Methods, DENV Reference 90 Database). The BLAST results are first cleaned to get the best hit for each sequence in the database, 91 based on alignment length, similarity, e-value and number of gaps. The tool determines which 92 reference sequence is more closely related to the query based on the identity and length of the 93 sequence covered, returning the serotype and genotype of the reference sequence for the purpose of 94 classifying the query.If a complete CDS fails to be obtained through the assembly process, the processed reads are 96 mapped against the DENV typing database, with Bowtie2 [13], using Seq Typing tool, with similar 97 criteria for coverage and identity to those used with the BLAST approach. If a type is determined, 98 the consensus sequence obtained follows through to the next step in the workflow. Otherwise, the 99 sample is classified as Non-Typable and its process terminated. 100 3.5 5. Phylogeny 101 All DENV complete CDSs and consensus sequences analysed in a workflow execution are aligned 102 with MAFFT [20] (version 7.402), in auto mode and with orientation being automatically determined 103 and adjusted. With the resulting alignment, a Maximum Likelihood tree is inferred with RaXML [21] 104 (version 8.2.11), using as default the GTR-Γ substitution model and a 500 times bootstrap. 105 Optionally, the closest reference sequence in the DENV typing database to each analysed sample 106 can be retrieved and included in the phylogenetic analysis. 107 4 Workflow Execution 108 The DEN-IM workflow can be executed in any UNIX-based system, from local machines to high-109 performance computing clusters (HPC) facilities with Nextflow and a container engine installation, 110 such as Docker (https://www.docker.com/, Shifter [22] or Singularity [23]. 111 Due to its Nextflow implementation, DEN-IM allows for out-of-the-box high-level parallelization 112 and offers direct support for distributed computational environments. DEN-IM integrates Docker 113 containerised images for all the tools necessary for its execution, ensuring reproducibility and the 114 tracking of both software code and version, regardless of the operating system used. The Docker 115 images provided are also compatible with other container engines. 116 Users can customise the workflow execution either by using command line options or by modifying 117 a simple plain-text configuration file (params.config). The version of each of the tools used in 118 DEN-IM can be changed by providing new container tags in the appropriate configuration file 119 (containers.config). The resources for each process can also be changed (resources.config). To make 120 the execution of the workflow as simple as possible, a set of default parameters and directives is 121 provided. 122 The local installation of the DEN-IM workflow, including the docker containers with all the tools 123 needed and the curated DENV database, requires 15 Gigabytes of free disk space. The minimum 124 requirements to execute the workflow are at least 5 Gigabytes of memory and 4 CPUs, although 7 125 Gigabytes of memory is advised. The disk space required for execution depends greatly on the size 126 of the input data, but for the datasets used in this article DEN-IM generates approximately 20 Gb 127 data per Gb of input data. 128 5 Output and Report 129 The output files of all tools are stored in the 'results' folder in the directory of the DEN-IM execution.130 The execution log file for each component, as well as a log file for the execution of the workflow, are 131 also available. 132 DEN-IM creates an interactive HTML report (Figure S1), stored in the 'pipeline results' directory, 133 containing all the information in the results divided into four sections: report overview, tables, charts 134 and phylogenetic tree. The report can be easily exchanged between collaborators by compressing 135 and sharing the \"pipeline report\" folder. 136 The report overview contains information about the number of samples in the analysis. It allows 137 for selection, filtering and highlighting of particular samples and tools in the workflow. 138 The table section contains the results and statistics the quality control, assembly, read mapping 139 and in silico typing results. The in silico typing table contains the results of the serotype and 140 genotype of each CDS analysed, as well as identity and coverage and GenBank ID for the closest 141 reference in the DENV typing database. doi: bioRxiv preprint 206 was generated. For the assembled CDSs, all but one were assembled with MEGAHIT after not 207 producing a full CDS with SPAdes. Moreover, pronounced variation on the size of the assembled 208 8 . CC-BY 4.0 doi: bioRxiv preprint As a positive control sample, the supernatant of a viral culture containing DENV-2 strain 16681 388 was used. The negative control sample consisted of DNA-and RNA-free water (Sigma-Aldrich, St. 389 Louis, MO, USA). 390 A spiked sample was produced consisting of a mixture of four 5 µl of cDNA isolated from clinical 391 samples including all DENV serotypes (DENV-1 to -4). The viral cDNA for these samples was not in 392 equal concentration and the viral copy number in the clinical samples was assessed by RT-PCR [35].393The results were as follow: DENV-2 with 1070000 copies/µl, DENV-1 with 117830 copies/µl, DENV-3 394 with 44300 copies/µl and DENV-4 with 6600 copies/µl.The cDNA libraries were generated using either the NEBNext R RNA First and Second strand 396 modules and the Nextera XT DNA library preparation kit (NXT), or the TruSeq RNA V2 library 397 preparation kit (TS). The libraries were sequenced in MiSeq and NextSeq instruments using 300-cycles 398 13 . CC-BY 4.0 International license author/funder. It is made available under aThe copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/628073 doi: bioRxiv preprint v2 paired-end cartridges.The DEN-IM workflow was executed with the raw sequencing data using the default parameters 400 and resources in an HPC cluster with 300 Cores/600 Threads of Processing Power and 3 TB RAM 401 divided through 15 computational nodes, 9 with 254 GB Ram and 6 with 126GB RAM. 402 9.4 Targeted Metagenomics Sequencing Data 403 The accession numbers for the 106 DENV-3 amplicon sequencing paired-end short-read datasets 404 are available under BioProject PRJNA394021. The list of Run Accession IDs were obtained 405 with NCBI's RunSelector and the raw data was downloaded with the GetSeqENA tool (https: 406 //github.com/B-UMMI/getSeqENA).The DEN-IM workflow was executed with the raw sequencing data with default parameters and 408 resources in the same HPC cluster as the shotgun metagenomics dataset. 409 10 Availability of source code and requirements 410 Lists the following: 411 • Project name: DEN-IM 412 • Project home page: https://github.com/B-UMMI/DEN-IM 413 • Operating system(s): UNIX-like systems. 414 • Programming language: Nextflow, Python, Bash 415 • Other requirements: Java version 8 or highest. Docker/Singularity/Shifter 416 • License: GNU GPL v3 417 • Documentation and tutorials: https://github.com/B-UMMI/DEN-IM/wiki 418 11 Availability of supporting data and materials 419 The 106 DENV-3 targeted metagenomics sequencing paired-end short-read datasets are available 420 under BioProject PRJNA394021. The 25 shotgun metagenomics dataset is available under Bioproject 421 PRJNA474413 The accession number for all the samples in the shotgun metagenomics dataset are 422 available in the supplemental material (Table S3) . 423 12 Declarations 424 12.1 List of abbreviations 425 Control 428 12.2 Ethical Approval (optional) 429 This study followed international standards for the ethical conduct of research involving human 430 subjects. Data and sample collection was carried out within the DENVEN and IDAMS (International 431 Research Consortium on Dengue Risk Assessment, Management and Surveillance) projects. The432 study was approved by the Ethics Review Committee of the Biomedical Research Institute, Carabobo 433 University (Aval Bioetico #CBIIB(UC)-014 and CBIIB-(UC)-2013-1), Maracay, Venezuela; the\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Health planners use forecasts of key metrics associated with influenza-like-illness (ILI); near-term weekly incidence, week of season onset, week of peak, and intensity of peak. Here, we describe our participation in a weekly prospective ILI forecasting challenge for the United States for the 2016-17 season and subsequent evaluation of our performance. We implemented a metapopulation model framework with 32 model variants. Variants differed from each other in their assumptions about: the force-of-infection (FOI); use of uninformative priors; the use of discounted historical data for not-yet-observed time points; and the treatment of regions as either independent or coupled. Individual model variants were chosen subjectively as the basis for our weekly forecasts; however, a subset of coupled models were only available part way through the season. Most frequently, during the 2016-17 season, we chose; FOI variants with both school vacations and humidity terms; uninformative priors; the inclusion of discounted historical data for not-yet-observed time points; and coupled regions (when available). Our near-term weekly forecasts substantially over-estimated incidence early in the season when coupled models were not available. However, our forecast accuracy improved in absolute terms and relative to other teams once coupled solutions were available. In retrospective analysis, we found that the 2016-17 season was not typical: on average, coupled models performed better when fit without historically augmented data. Also, we tested a simple ensemble model for the 2016-17 season and found that it underperformed our subjective choice for all forecast targets. In this study, we were able to improve accuracy during a prospective forecasting exercise by coupling dynamics between regions. Although reduction of forecast subjectivity should be a long-term goal, some degree of human intervention is likely to improve forecast accuracy in the medium-term in parallel with the systematic consideration of more sophisticated ensemble approaches.It is estimated that there are between 3 and 5 million worldwide annual seasonal cases 1 of severe influenza illness, and between 290 000 and 650 000 respiratory deaths [1].2 Influenza-like-illness (ILI) describes a set of symptoms and is a practical way for 3 health-care workers to easily estimate likely influenza cases. The Centers for Disease 4 Control (CDC) collects and disseminates ILI information, and has, for the last several 5 years, run a forecasting challenge (the CDC Flu Challenge) for modelers to predict 6 near-term weekly incidence, week of season onset, week of peak, and intensity of peak. 7 We have developed a modeling framework that accounts for a range of mechanisms 8 thought to be important for influenza transmission, such as climatic conditions, school 9 vacations, and coupling between different regions. In this study we describe our forecast 10 procedure for the 2016-17 season and highlight which features of our models resulted in 11 better or worse forecasts. Most notably, we found that when the dynamics of different 12 regions are coupled together, the forecast accuracy improves. We also found that the 13 most accurate forecasts required some level of forecaster interaction, that is, the 14 procedure could not be completely automated without a reduction in accuracy.PLOS 2/43 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309021 doi: bioRxiv preprint Infectious pathogens with short generation times pose public health challenges because 17 they generate substantial near-term uncertainty in the risk of disease. This uncertainty 18 is most acute and shared globally during the initial stages of emergence of novel human 19 pathogens such as SARS [2], pandemic influenza [3], or the Zika virus [4] . However, at 20 national and sub-national levels, uncertainty arises frequently for epidemic pathogens 21 such as seasonal influenza, dengue, RSV and rotavirus; causing problems both for 22 health planners and at-risk individuals who may consider changing their behavior to 23 mitigate their risk during peak periods. 24 Seasonal influenza affects populations in all global regions and is forecast annually in 25 temperate populations, either implicitly or explicitly [5]. Peak demand for both 26 outpatient and inpatient care is driven by peak incidence of influenza in many years [6]. 27 Therefore, the efficient provision of elective procedures and other non-seasonal health 28 care can be improved by accurate forecasts of seasonal influenza. Implicitly, most 29 temperate health systems use knowledge of historical scenarios with which to plan for 30 their influenza season. The current situation is then assessed against the deviation from 31the historical averages and worst-cases as observed in their own surveillance system.The United States Centers for Disease Control (CDC) has sought to formalize regional 33 and national forecasts by introducing an annual competition [7] . Each week, 34 participating teams submit weekly estimates of incidence for the next four weeks, season 35 onset, and timing and intensity of the peak. Methods used by teams include purely 36 statistical models, [8-10] mechanistic models [11, 12] machine learning and hybrid 37 approaches [13-15]. Expert-opinion surveys have also been used and performed well.38 Some teams augment their forecasts of the official ILI data with the use of potentially 39 faster or less-noisy datasets such as google flu trends [16].40 Here we describe a our mechanistic-model-supported participation in the 2016-17 CDC 41 influenza forecasting challenge, as an example of a disease forecasting process. We 42 emphasize a subjective human component of this process and also describe a 43 retrospective evaluation of the models for the previous six seasons. All the models 44 PLOS 3/43 . CC-BY 4.0 International license is made available under aThe copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309021 doi: bioRxiv preprint described are implemented in the R package Dynamics of Interacting Community 45 Epidemics (DICE, https://github.com/predsci/DICE). 46 Methods 47 Data 48 The CDC Influenza-like-illness Surveillance Network (ILINet) Human and Health 49 Services (HHS) region and national data were downloaded from the CDC-hosted web 50 application FluView [17] and used to create a historic database of ILI cases. Fig S1 51shows which states are grouped into each HHS region, along with the population of each 52 region. Because we require an absolute number of cases per week, the CDC ILINet data 53 is converted from percent ILI cases per patient to ILI cases. We estimate the absolute 54 number of weekly ILI cases by dividing the weighted percent of ILI cases in the CDC 55 data by 100 and multiplying it by the total weekly number of patients. We assume two 56 outpatient visits per person per year so that the total weekly number of patients is 57 estimated as: (total regional population)x(2 outpatient visits per person per year)x(1 58 year/52 weeks). 59 The estimate of two outpatient visits per year is based on two studies. In 2006 60 Schappert and Burt [18] studied the National Ambulatory Medical Care Surveys 61 (NAMCS) and the National Hospital Ambulatory Medical Care Surveys (NHAMCS) 62 and calculated an ambulatory rate of 3.8 visits per capita-year. The 2011 NAMCS [19] 63 and NHAMCS surveys [20] estimated ambulatory visit rates of 3.32 visits per capita per 64year for physician's offices, 0.43 for hospital emergency departments and 0.33 hospital 65 Outpatient Departments. We sum these rates to get an outpatient visit per capita-year 66 of 4.08. We further estimate from the surveys that only half of these outpatient clinics 67 are sites that report to ILINet, and hence we rounded to our two outpatient visits per 68 year estimate. 69 Specific humidity (SH) is measured in units of kg per kg and is defined as the ratio of 70 PLOS 4/43 . CC-BY 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/309021 doi: bioRxiv preprint water vapor mass to total moist air mass. Two other measurements of humidity are 71 absolute humidity and relative humidity. SH is included in DICE as a potential modifier 72 of transmissibility for this time period and uses Phase-2 of the North American Land 73 Data Assimilation System (NLDAS-2) data base provided by NASA [21-23]. The 74 NLDAS-2 data base provides hourly specific humidity (measured 2-meters above the 75 ground) for the continental US at a spatial grid of 0.125 • which we average to daily and 76 weekly values. The weekly data is then spatially-averaged for the states and CDC 77 regions. 78 School vacation schedules were collected for the 2014-15 and 2015-16 academic years for 79 every state. For each state, a school district was identified to represent each of the three 80 largest cities. Vacation schedules were then collected directly from the district websites. 81 These three school vacation schedules were first processed to a weekly schedule with a 82 value of 0 indicating class was in session all five weekdays and a 1 indicating five 83 vacation days. Next, the representative state schedule was produced by averaging the 84 three weekly district schedules. Region schedules are obtained by a population-weighted 85 average of the state schedules. Similarly, the national schedule is generated by a 86 population-weighted average of the regions. 87 For the 2016-17 season we determine start and end times as well as spring and fall 88 breaks from the previous years schedules. Thanksgiving and winter vacation timing was 89 taken from the calendar where the winter break is assumed to be the last two calendar 90 weeks of the year. Based on the proportion of schools closed and number of days closed, 91 p(t) is assigned a value in the range [0, 1]. For example in week t i , if all schools are 92 closed for the entire week then we define the proportion of open schools p(t i ) = 1. 93 However, if all schools have Monday and Tuesday off (missing 2 of 5 days), then 94 p(t i ) = 0.4. Similarly, if 3 of 10 schools have spring break (entire week off), but the 95 other 7 schools have a full week of class then p(t i ) = 0.3. If all schools have a full week 96 of class then p(t i ) = 0.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The host antiviral response involves the induction of interferons and proinflammatory cytokines, but also the activation of cell death pathways, including apoptosis, to limit viral replication and spreading. This host defense is strictly regulated to eliminate the infection while limiting tissue damage that is associated with virus pathogenesis. Post-translational modifications, most notably phosphorylation, are key regulators of the antiviral defense implying an important role of protein phosphatases. Here, we investigated the role of the dual-specificity phosphatase 1 (DUSP1) in the host defense against human respiratory syncytial virus (RSV), a pathogenic virus of the Pneumoviridae family, and Sendai virus (SeV), a model virus being developed as a vector for anti-RSV vaccine. We found that DUSP1 is upregulated before being subjected to proteasomal degradation. DUSP1 does not inhibit the antiviral response, but negatively regulates virus-induced JNK/p38 MAPK phosphorylation. Interaction with the JNK-interacting protein 1 scaffold protein prevents dephosphorylation of JNK by DUSP1, likely explaining that AP-1 activation and downstream cytokine production are protected from DUSP1 inhibition.Importantly, DUSP1 promotes SeV-induced apoptosis and suppresses cell migration in RSVinfected cells. Collectively, our data unveil a previously unrecognized selective role of DUSP1 in the regulation of tissue damage and repair during infections by RSV and SeV.. CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/163360 doi: bioRxiv preprint relies on a complex set of phosphorylations mainly mediated by the TANK-binding kinase 1 (TBK1)/ IκB kinase epsilon (IKKε) kinases 13-17 . NF-κB, mainly p65/p50, activation during SeV and RSV infections occurs through IκB kinase (IKK)-dependent phosphorylation of the NF-κB inhibitor protein IκBα and of the p65 subunit 18, 19 . The signaling cascade leading to AP-1 activation is more elusive, but ultimately results in the phosphorylation of the nuclear Activating Transcription Factor 2 (ATF-2) and c-Jun subunits by JNK and p38 Mitogen-Activated Protein Kinases (MAPKs) 10,20,21 . The activation of these transcription factors promotes the transcription of early antiviral genes, type I/III IFNs and proinflammatory cytokines and chemokines 12, 22, 23 . In response to IFNs, hundreds of interferon-stimulated genes (ISGs) are induced to limit virus replication through enhancement of virus detection and innate immune signaling, cytoskeleton remodelling, inhibition of protein translation, induction of apoptosis, amongst other antiviral functions 24-26 . These same PRRs have also been shown to activate IFN-independent cell death pathways, including apoptosis 8 .Post-translational modifications (PTMs), including phosphorylation, ubiquitylation and acetylation, of signal transduction proteins involved in the pathways engaged downstream of virus sensing are crucial to reach a fine-tuned regulation of the innate immune response 27 . In light of the very well documented importance of phosphorylation PTMs in the regulation of the antiviral response, the protein phosphatases negatively regulating the signaling events have started to be identified. The exact role of the Ser/Thr protein phosphatase 1 (PP1) in the antiviral response remains elusive as PP1α and PP1γ were found to dephosphorylate MDA-5 and RIG-I leading to their activation 28 , while they were also described to be responsible for the dephosphorylation of key C-terminal phosphoresidues of IRF3 leading to its inhibition 29 . Most . CC-BY-NC-ND 4.0 International license is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. It . https://doi.org/10.1101/163360 doi: bioRxiv preprint 15 Clement, J. F. et al. Phosphorylation of IRF-3 on Ser 339 generates a hyperactive form of IRF-3 through regulation of dimerization and CBP association.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "are Co-corresponding authorsWord Counts: 2574 words Abstract Background Previous studies had described the differences in clinical characteristics between ICU and non-ICU patients. However, seldom study focused on confirmed and unconfirmed groups. Our aim was to compare clinical and imaging characteristics of COVID-19 patients outside Hubei province between confirmed and unconfirmed group.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The outbreak of pneumonia caused by a novel coronavirus (2019-nCoV) in Wuhan 17City of China obtained global concern, the population outflow from Wuhan has 18 contributed to spatial expansion in other parts of China. We examined the effects of 19 population outflow from Wuhan on the 2019-nCoV transmission in other provinces 20 and cities of China, as well as the impacts of the city closure in Wuhan. We observed 21 a significantly positive association between population movement and the number of 22 cases. Further analysis revealed that if the city closure policy was implemented two 23 days earlier, 1420 (95% CI: 1059, 1833) cases could be prevented, and if two days 24 later, 1462 (95% CI: 1090, 1886) more cases would be possible. Our findings suggest 25 that population movement might be one important trigger of the 2019-nCoV infection 26 transmission in China, and the policy of city closure is effective to prevent the 27 epidemic. 28\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "In 2013, U.S. swine producers were confronted with the disruptive emergence of porcine epidemic diarrhoea (PED). Movement of animals among farms is hypothesised to have played a role in the spread of PED among farms. Via this or other mechanisms, the rate of spread may also depend on the geographic density of farms and climate. To evaluate such effects on a large scale, we analyse state-level counts of outbreaks with variables describing the distribution of farm sizes and types, aggregate flows of animals among farms, and an index of climate. Our first main finding is that it is possible for a correlation analysis to be sensitive to transmission model parameters. This finding is based on a global sensitivity analysis of correlations on simulated data that included a biased and noisy observation model based on the available PED data. Our second main finding is that flows are significantly associated with the reports of PED outbreaks. This finding is based on correlations of pairwise relationships and regression modeling of total and weekly outbreak counts. These findings illustrate how variation in population structure may be employed along with observational data to improve understanding of disease spread.together produce 95 percent of the U.S. pig crop. 5 Farms experiencing outbreaks have suffered 90 percent and higher losses of . CC-BY 4.0 International license is made available under a\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "confirmed and 1368 patients have died from the disease. However, the clinical characteristics of the dyed patients were still not clearly clarified. This study aims to summarize the clinical characteristics of death cases with COVID-19 and to identify critically ill patients of COVID-19 early and reduce their mortality.The clinical records, laboratory findings and radiologic assessments included chest X-ray or computed tomography were extracted from electronic medical records of 25 died patients with COVID-19 in Renmin Hospital of Wuhan University from Jan 14 to Feb 13, 2020. Two experienced clinicians reviewed and abstracted the data.The mean age of the dead was 71.48 ± 12.42 years, the average course of the disease was 10.56 ± 4.42 days, all patients eventually died of respiratory failure. All of those who died had underlying diseases, the most common of which was hypertension (16/25, 64%), followed by diabetes (10/25, 40%), heart diseases (8/25, 32%), kidney diseases (5/25, 20%), cerebral infarction (4/25, 16%), chronic obstructive pulmonary disease (COPD, 2/25, 8%), malignant tumors (2/25, 8%) and acute pancreatitis (1/25, 4%). The most common organ damage outside the lungs was the heart, followed by kidney and liver. In the patients' last examination before death, white blood cell and neutrophil counts were elevated in 17 patients (17/25, 68%) and 18 patients (18/25, 72%), lymphocyte counts were decreased in 22 patients (22/25, 88%). Most patients' PCT, CRP and SAA levels were elevated, the percentages were 90.5% (19/21), 85% (19/20) and 100% (21/21) respectively. The levels of the last test of neutrophils (15/16, 93.8%), PCT (11/11, 100%), CRP (11/13, 84.6%), cTnI (8/9, 88.9%), D-Dimer (11/12, 91.6%) and LDH (9/9, 100%) were increased as compared to the first test, while the levels of lymphocytes were decreased (14/16, 87.5%).Interpretation The age and underlying diseases (hypertension, diabetes, etc.) were the most All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The 2019 novel coronavirus (2019-nCoV or SARS-CoV-2) has spread more rapidly than any other betacoronavirus including SARS-CoV and MERS-CoV.However, the mechanisms responsible for infection and molecular evolution of this virus remained unclear. 5 Methods: We collected and analyzed 120 genomic sequences of 2019-nCoV including 11 novel genomes from patients in China. Through comprehensive analysis of the available genome sequences of 2019-nCoV strains, we have tracked multiple inheritable SNPs and determined the evolution of 2019-nCoV relative to other coronaviruses. 10 Results: Systematic analysis of 120 genomic sequences of 2019-nCoV revealed cocirculation of two genetic subgroups with distinct SNPs markers, which can be used to trace the 2019-nCoV spreading pathways to different regions and countries. Although 2019-nCoV, human and bat SARS-CoV share high homologous in overall genome structures, they evolved into two distinct groups with different receptor entry specificities 15through potential recombination in the receptor binding regions. In addition, 2019-nCoV has a unique four amino acid insertion between S1 and S2 domains of the spike protein, which created a potential furin or TMPRSS2 cleavage site.Our studies provided comprehensive insights into the evolution and spread of the 2019-nCoV. Our results provided evidence suggesting that 2019-nCoV 20 may increase its infectivity through the receptor binding domain recombination and a cleavage site insertion. author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "To compare the sex differences in the clinical findings among patients with severe coronavirus disease 2019 .We retrospectively collected data of 47 patients diagnosed as severe type of COVID-19 from February 8 to 22, 2020, including demographics, illness history, physical examination, laboratory test, management, and compared differences between men and women.: Of the 47 patients, 28 (59.6%) were men. The median age was 62 years, and 30 (63.8%) had comorbidities. The initial symptoms were mainly fever (34 [72.3%]), cough (36 [76.6%]), myalgia (5 [10.6%]) and fatigue (7 [14.9%]). Procalcitonin level was higher in men than in women (0.08 vs.0.04ng/ml, p=0.002). N-terminal-pro brain natriuretic peptide increased in 16 (57.1%) men and 5 (26.3%) women (p=0.037). Five men (17.9%) had detected positive influenza A antibody, but no women. During 2-week admission, 5 (17.9%) men and 1 (5.3%) woman were reclassified into the critical type due to deterioration. Mortality was 3.6% in men and 0 in women respectively. Four (21.1%) women and one man (3.6%) recovered and discharged from hospital.Conclusion: Sex differences may exist in COVID-19 patients of severe type.Men are likely to have more complicated clinical condition and worse in-hospital outcomes as compared to women. : medRxiv preprint 0.508 Neutrophil count, ×10 9 /L 3.1 (2.1-5.2) 4.1 (2.0-6.9) 2.9 (2.5-3.4) 0.353Lymphocyte count, ×10 9 /L 1.3 (0.7-1.9) 1.2 (0.7-2.0) 1.3 (0.9-1.6) 0869 ≥1.0, No. (%) 30 (63.8) 15 (53.6) 15 (78.9) 0.076 Platelet count, ×10 9 /L 230.0 (157.0-302.5) 218.0 (151.3-320.8) 230.0 (164.0-304.0) 0.696 ＞300.0, No. (%) 14 (29.8) 8 (28.6) 6 (31.6) 0.825 Procalcitonin, ng/ml 0.06 (0.03-0.10) 0.08 (0.05-0.19) 0.04 (0.03-0.07) 0.002 increased (＞0.05), No. (%) 25 (53.2) 19 (67.9) 6 (31.6) 0.014 All rights reserved. No reuse allowed without permission.the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "This study shows that the disease free equilibrium ( ) for COVID-19 coronavirus does not satisfy the criteria for a locally or globally asymptotic stability. This implies that as a pandemic as declared by WHO (2020) the COVID-19 coronavirus does not have a curative vaccine yet and precautionary measures are advised through quarantine and observatory procedures. Also, the Basic Reproductive number ( 0 < 1) by Equation (33) shows that there is a chance of decline of secondary infections when the ratio between the incidence rate in the population and the total number of infected population quarantined with observatory procedure.The effort to evaluate the disease equilibrium shows that unless there is a dedicated effort from government, decision makers and stakeholders, the world would hardly be reed of the COVID-19 coronavirus and further spread is eminent and the rate of infection will continue to increase despite the increased rate of recovery because of the absence of vaccine at the moment.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Motivation: Understanding the relationship between the sequence, structure, binding energy, binding kinetics and binding thermodynamics of protein-protein interactions is crucial to understanding cellular signaling, the assembly and regulation of molecular complexes, the mechanisms through which mutations lead to disease, and protein engineering. Results: We present SKEMPI 2.0, a major update to our database of binding free energy changes upon mutation for structurally resolved protein-protein interactions. This version now contains manually curated binding data for 7085 mutations, an increase of 133%, including changes in kinetics for 1844 mutations, enthalpy and entropy changes for 443 mutations, and 440 mutations which abolish detectable binding. Availability:The database is available at\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Members of the Nidovirales order have (+)RNA genomes amongst the largest in size in the RNA virus world. Expression of their genes is promoted through reading of genomic RNA and mRNA transcripts by the ribosome of the infected cell. The 5'-end of these RNAs is supposedly protected by an RNA-cap structure (m7GpppNm) whose most synthesis steps remain elusive. In Eukaryotes, the RNA-cap structure is methylated by RNA methyltransferases (MTases) at the RNA-cap N7-guanine position as well as the 2'-O methyl position of the first transcribed nucleotide. In Coronaviridae, two separate enzymes (nsp14 and nsp16) perform the N7-guanine and the 2'-OH methylation, respectively. One salient feature of the Nidovirales N7-guanine MTase nsp14 is that it is the only example of non-Rossman fold viral MTase known so far. Conversely, all other Nidovirales nsp16-like MTases have a canonical Rossman fold. Many Nidovirales members lack either any RNA MTase signature sequence (e.g., Arteriviridae), or lack a N7-guanine MTase signature sequence (e.g., Tobaniviridae, Euroniviridae, Roniviridae, Medioniviridae). Both nsp14-and nsp16-like enzyme genes are usually located in Orf1b encoding for the replication machinery. Here, we report the discovery of a putative Rossman fold RNA MTase in the Orf1a of ten Tobaniviridae members. Multiple sequence alignments and structural analyses identify this novel gene as a typical RNA-cap N7-guanine MTase with substrate specificity and active-site organization similar to the canonical eukaryotic RNA-cap N7-guanine MTase.author/funder. All rights reserved. No reuse allowed without permission.The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/639369 doi: bioRxiv preprint Up to recently, four viral families, Arteriviridae, Coronaviridae, Roniviridae, and Mesoniviridae constituted the Nidovirales order. The Coronaviridae family was further divided in two subfamilies, Coronavirinae and Torovirinae. The Torovirinae family contained two genus Torovirus and Bafinivirus, plus few unassigned species. Recently, the ICTV proposed a novel classification into the Nidovirales order, with the creation of nine families [1] . In this novel ICTV proposal, ratified on February 2019, Coronaviridae do not encompass both Coronavirinae and Toronavirinae subfamilies. Rather, the new suborder Tornidovirinae (including toroviruses and bafiniviruses) contains a novel family, Tobaniviridae, at the same level as the Coronaviridae, which is now within suborder Cornidovirineae. This new taxonomy is used here, as depicted in Fig. 1 .Nidovirales have attracted much attention in the past decades for being causative agents of serious diseases in humans and animals. In humans, Severe Acute Respiratory Syndrome (SARS) and the Middle-East Respiratory Syndrome (MERS) coronaviruses (CoVs) have been associated with high fatality/case ratios of ∼10% and ∼35 %, respectively (reviewed in [2]). In animals, pathogens of significant economical and societal impact include Equine Arteritis virus (EAV, Alphaarterivirus equid) and Porcine Reproductive Respiratory Syndrome virus (PRSSV 1-2, Betaarterivirus suid 1-2).Two salient features of these viruses are, amongst others, their complex and still obscure replication/transcription mechanism as well as their broad genome-size range. Genomes of Arteriviridae (12.7 -15.7 kb in length), Mesoniviridae (∼20 kb), Medioniviridae (20.2 -25 kb), Euroniviridae (∼24.5 kb), Roniviridae (∼26 kb) Tobaniviridae and Coronaviridae (∼27-32 kb), Abyssoviridae (35.9 kb), and Mononiviridae (41.1 kb) members, share a similar genomic organisation generating viral mRNAs from a fixed genomic location, the 'nest', which has inspired the family name (Nido : latin for 'nest') [3, 4] . Nidovirales genomic and mRNAs are presumably capped, although direct demonstration of the presence of an RNA cap structure is still missing for many of these Nidovirales members.In Eukaryotes, the RNA cap structure is thought to be synthesized post-transcriptionally [5] .Most of (+)RNA viruses supposedly follow a conventional RNA capping pathway [6] in which nascent viral 5'-triphosphate genomic (or mRNAs) is thought to be processed through three enzymatic reactions to yield an RNA cap whose structure is indistinguishable from that of cellular mRNAs. The capping pathway implies that the 5'-triphosphate RNA is hydrolyzed into 5'-diphosphate by an RNA triphosphatase, a GMP residue (the « cap », coming from GTP) is next covalently transferred to the 5'-diphosphate RNA in the 5' to 5' orientation by a guanlylyltransferase (GTase), releasing inorganic pyrophosphate. Both the cap and the first transcribed nucleotide are then methylated at the N7-guanine (mGpppN-RNA, so-called Cap0 structure) and the 2'-oxygen position (mGpppNm-RNA, Cap1 structure) by one or two RNA methyltransferases (MTases), respectively. The MTase uses S-Adenosyl-Methionine (SAM) as the methyl donor and releases S-Adenosyl-homocysteine (SAH).There is no high-resolution structural analysis available of Nidovirales RNA caps. The evidence for their presence comes from immunological detection in Torovirus genomic and mRNAs [7] , and co-migration analysis in low-resolution chromatographic techniques for MHV [8] and Simian Hemorragic fever virus [9] . In other instances, Nidovirales cap structures supposedly conform to the canonical eukaryotic RNA capping pathway because enzymes usually deemed necessary to perform RNA capping are present in cognate genomes (reviewed in [6]). Bioinformatic analyses have been used to detect signature-sequences typically associated with RNA capping/modifying enzymes i.e guanlylyltransferase (GTase), polymerase, helicase, nuclease, and methyltransferase (MTase). These signature-sequences have served in many instances to isolate corresponding enzymes endowed with the predicted activity. In Nidovirales, this approach has been pioneered for Coronavirus enzymes, yet author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "A pattern in which nucleotide transitions are favored several-fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins, measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1239 replacements, the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval, 50 % to 56 %), compared to the null expectation of 50 %. We show that this effect is not large compared to that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, available data have the power to verify the \"conservative transitions\" hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias.. CC-BY 4.0 International license is made available under a\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Pompe disease results from a defect in human acid α-glucosidase (GAA), a lysosomal enzyme 17 that cleaves terminal α1-4 and α1-6 glucose from glycogen. In Pompe disease (also known as 18 Glycogen Storage Disorder type II), the accumulation of undegraded glycogen in lysosomes 19 leads to cellular dysfunction, primarily in muscle and heart tissues. Pompe disease is an active 20 candidate of clinical research, with pharmacological chaperone therapy tested and enzyme 21 replacement therapy approved. Despite production of large amounts of recombinant GAA 22 annually, the structure of GAA has not been reported until now. Here, we describe the first 23 structure of GAA, at 1.7Å resolution. Three structures of GAA complexes reveal the molecular 24 basis for the hundreds of mutations that lead to Pompe disease and for pharmacological 25 chaperoning in the protein. The GAA structure reveals a surprising second sugar-binding site 26\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Background Since December 2019, a cluster of coronavirus disease 2019 (COVID-19) occurred in Wuhan, Hubei Province, China and spread rapidly from China to other countries. In-hospital mortality are high in severe cases and cardiac injury characterized by elevated cardiac troponin are common among them. The mechanism of cardiac injury and the relationship between cardiac injury and in-hospital mortality remained unclear. Studies focused on cardiac injury in COVID-19 patients are scarce.Objectives To investigate the association between cardiac injury and in-hospital mortality of patients with confirmed or suspected COVID-19.Methods Demographic, clinical, treatment, and laboratory data of consecutive confirmed or suspected COVID-19 patients admitted in Wuhan No.1 Hospital from 25 th December, 2019 to 15 th February, 2020 were extracted from electronic medical records and were retrospectively reviewed and analyzed. Univariate and multivariate Cox regression analysis were used to explore the risk factors associated with in-hospital death.A total of 110 patients with confirmed (n=80) or suspected (n=30) COVID-19 were screened and 48 patients (female 31.3%, mean age 70.58±13.38 year old) among them with high-sensitivity cardiac troponin I (hs-cTnI) test within 48 hours after admission were included, of whom 17 (17/48, 35.4%) died in hospital while 31 (31/48, 64.6%) were discharged or transferred to other hospital. High-sensitivity cardiac troponin I was elevated in 13 (13/48, 27.1%) patents. Multivariate Cox regression analysis showed pulse oximetry of oxygen saturation (SpO2) on admission (HR 0.704, 95% CI 0.546-0.909, per 1% decrease, p=0.007), elevated hs-cTnI (HR 10.902, 95% 1.279-92.927, p=0.029) and elevated d-dimer (HR 1.103, 95%CI 1.034-1.176, per 1mg/L increase, p=0.003) on admission were independently associated with in-hospital mortality.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Rapid spread of SARS-CoV-2 in Wuhan prompted heightened surveillance in Shenzhen and elsewhere in China. The resulting data provide a rare opportunity to measure key metrics of disease course, transmission, and the impact of control.The Shenzhen CDC identified 391 SARS-CoV-2 cases from January 14 to February 12, 2020 and 1286 close contacts. We compare cases identified through symptomatic surveillance and contact tracing, and estimate the time from symptom onset to confirmation, isolation, and hospitalization. We estimate metrics of disease transmission and analyze factors influencing transmission risk.Cases were older than the general population (mean age 45) and balanced between males (187) and females (204). Ninety-one percent had mild or moderate clinical severity at initial assessment. Three have died, 225 have recovered (median time to recovery is 32 days). Cases were isolated on average 4.6 days after developing symptoms; contact tracing reduced this by 1.9 days. Household contacts and those travelling with a case where at higher risk of infection (ORs 6 and 7) than other close contacts. The household secondary attack rate was 15%, and children were as likely to be infected as adults. The observed reproductive number was 0.4, with a mean serial interval of 6.3 days.Our data on cases as well as their infected and uninfected close contacts provide key insights into SARS-CoV-2 epidemiology. This work shows that heightened surveillance and isolation, particularly contact tracing, reduces the time cases are infectious in the community, thereby reducing R . Its overall impact, however, is uncertain and highly dependent on the number of asymptomatic cases. We further show that children are at similar risk of infection as the general population, though less likely to have severe symptoms; hence should be considered in analyses of transmission and control.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The bacterial bipartite transferrin receptor is an iron acquisition system that is required for 12 survival by several key human and animal pathogens. It consists of the TonB-dependent 13 transporter Transferrin binding protein A (TbpA) and the surface lipoprotein Transferrin binding 14 protein B (TbpB). Curiously, the Tbps are only found in host specific pathogens, and are 15 themselves host specific, meaning that they will bind to the transferrin of their host species, but 16 not to those of other animal species. While this phenomenon has long been established, neither 17 the steps in the evolutionary process that led to this exquisite adaptation for the host, nor the 18 steps that could alter it, are known. We sought to gain insight into these processes by studying 19Tbp specificity in Histophilus somni, a major pathogen of cattle. A past study showed that whole 20 cells of H. somni specifically bind bovine transferrin, but not transferrin from sheep and goats, 21 two bovids whose transferrins share 93% amino acid sequence identity with bovine transferrin. 22To our surprise, we found that H. somni can use sheep and goat transferrins as iron sources for 23 growth, and that HsTbpB, but not HsTbpA, has detectable affinity for sheep and goat 24 transferrins. Furthermore, a third transferrin binding protein, HsTbpA2, also showed affinity for 25 sheep and goat transferrins. Our results show that H. somni TbpB and TbpA2 act to broaden the 26 host transferrin recognition range of H. somni. 27\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Viruses are one of the major causes of various acute and chronic infectious diseases and thus a major contributor to the global burden of disease. Several studies have shown how viruses have evolved to hijack basic cellular pathways and evade innate immune response by modulating key host factors and signalling pathways. A collective view of these multiple studies could advance our understanding of viral evasion mechanisms and provide new therapeutic perspectives for the treatment of viral diseases. Here, we performed an integrative meta-analysis to elucidate the 17 different host-virus interactomes. Network and bioinformatics analyses showed how viruses with small genomes efficiently achieve the maximal effect by targeting multifunctional and highly connected host proteins with a high occurrence of disordered regions. We also identified the core cellular process subnetworks that are targeted by all the viruses. Integration with functional RNA interference (RNAi) datasets showed that a large proportion of the targets are required for viral replication. Furthermore, we performed an interactome-informed drug re-purposing screen and identified novel activities for broad-spectrum antiviral agents against hepatitis C virus and human metapneumovirus. Altogether, these orthogonal datasets could serve as a platform for hypothesis generation and follow-up studies to broaden our understanding of the viral evasion landscape. author/funder. All rights reserved. No reuse allowed without permission.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "An epidemic can be characterized by its speed (i.e., the exponential growth rate r) and strength (i.e., the reproductive number R). Disease modelers have historically placed much more emphasis on strength, in part because the effectiveness of an intervention strategy is typically evaluated on this scale. Here, we develop a mathematical framework for this classic, strength-based paradigm and show that there is a corresponding speed-based paradigm which can provide complementary insights. In particular, we note that r = 0 is a threshold for disease spread, just like R = 1, and show that we can measure the speed and strength of an intervention on the same scale as the speed and strength of an epidemic, respectively. We argue that, just as the strength-based paradigm provides the clearest insight into certain questions, the speed-based paradigm provides the clearest view in other cases. As an example, we show that evaluating the prospects of \"test-and-treat\" interventions against the human immunodeficiency virus (HIV) can be done more clearly on the speed than strength scale, given uncertainty in the proportion of HIV spread that happens early in the course of infection. We suggest that disease modelers should avoid over-emphasizing the reproductive number at the expense of the exponential growth rate, but instead look at these as complementary measures.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Sialic acids (Sia) are the primary receptors for influenza viruses, and are widely displayed on cell surfaces and in secreted mucus. Sia may be present in variant forms that include O-acetyl modifications at C4, C7, C8, and C9 positions, and N-acetyl or N-glycolyl at C5. They can also vary in their linkages, including α2-3 or α2-6-linkages. Here, we analyzed the distribution of modified Sia in cells and tissues of wild-type mice, or in mice lacking cytidine 5'-30 monophosphate-N-acetylneuraminic acid hydroxylase (CMAH) enzyme that synthesizes Nglycolyl modifications (Neu5Gc). We also examined the variation of Sia forms on erythrocytes and saliva from different animals. To determine the effect of Sia modifications on influenza A virus (IAV) infection, we tested for effects on hemagglutinin (HA) binding and neuraminidase (NA) cleavage. We confirmed that 9-O-acetyl, 7,9-O-acetyl, 4-O-acetyl, and Neu5Gc 35 modifications are widely but variably expressed in mouse tissues, with the highest levels detected in the respiratory and gastrointestinal tracts. Secreted mucins in saliva and surface proteins of erythrocytes showed a great degree of variability in display of modified Sia between different species. IAV HA from different virus strains showed consistently reduced binding to both Neu5Gc and O-acetyl modified Sia; however, while IAV NA were inhibited by Neu5Gc and 40 O-acetyl modifications, there was significant variability between NA types. The modifications of Sia in mucus may therefore have potent effects on the functions of IAV, and may affect both pathogens and the normal flora of different mucosal sites. (241 words) IMPORTANCE 45Sialic acids (Sia) are involved in many different cellular functions and are receptors for many pathogens. Sia come in many chemically modified forms but we lack a clear understanding of how they alter the interactions with microbes. Here we examine the expression of modified Sia in mouse tissues, on secreted mucus in saliva, and on erythrocytes, including those from IAV host species and animals used in IAV research. These Sia forms varied considerably between\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Since December 2019, Coronavirus Disease 2019 (COVID-19) emerged in Wuhan city and rapidly spread throughout China. The mortality of novel coronavirus pneumonia (NCP) in severe and critical cases is very high. Facing this kind of public health emergency, high efficient administrative emergency responsive mode in designated hospital is needed.All rights reserved. No reuse allowed without permission.author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Bayesian inference methods for individual-level geo-located epidemiological data have enabled reconstruction of transmission trees and quantification of disease spread in space and time, while accounting for uncertainty in missing data. However, these methods have rarely been applied to endemic diseases or ones in which asymptomatic infection plays a role, for which novel estimation methods are required. Here, we develop such methods to analyse longitudinal incidence data on visceral leishmaniasis (VL), and its sequela, postkala-azar dermal leishmaniasis (PKDL), in a highly endemic community in Bangladesh. Incorporating recent data on infectiousness of VL and PKDL, we show that while VL cases drive transmission when incidence is high, the contribution of PKDL increases significantly as VL incidence declines (reaching 55% in this setting). Transmission is highly focal: >85% of mean distances from inferred infectors to their secondary VL cases were <300m, and estimated average times from infector onset to secondary case infection were <4 months for 90% of VL infectors, but up to 2.75yrs for PKDL infectors. Estimated numbers of secondary VL cases per VL and PKDL case varied from 0-6 and were strongly correlated with the infector's duration of symptoms. Counterfactual simulations suggest that prevention of PKDL could have reduced VL incidence by up to a quarter. These results highlight the need for prompt detection and treatment of PKDL to achieve VL elimination in the Indian subcontinent and provide quantitative estimates to guide spatiotemporally-targeted interventions against VL.visceral leishmaniasis | post-kala-azar dermal leishmaniasis | spatiotemporal transmission | transmission tree | Bayesian inference S patiotemporal heterogeneity in incidence is a hallmark of 1 infectious diseases. Insight into this heterogeneity has in-2 creased considerably in recent years due to greater availability 3 of geo-located individual-level epidemiological data and the 4 development of sophisticated statistical inference methods for 5 partially observed transmission processes (1-6). These meth-6 ods have been developed for epidemics, in which the immune 7 status of the population is known, and for diseases with a 8 short time course that are relatively easily diagnosed, such 9 as measles, influenza, and foot-and-mouth disease (3, 4, 7).Here, we extend these methods to a slowly progressing endemic 11 disease of humans in which asymptomatic infection plays an 12 important role. 13 We analyse detailed longitudinal individual-level data on 14 incidence of visceral leishmaniasis (VL), and its sequela, post-15 kala-azar dermal leishmaniasis (PKDL), in a highly endemic 16 community in Fulbaria, Bangladesh (8). VL, also known as 17 kala-azar, is a lethal sandfly-borne parasitic disease targeted 18 for elimination as a public health problem (<1 case/10,000 19 people/year at subdistrict/district level depending on the 20 country) in the Indian subcontinent (ISC) by 2020 (9). PKDL 21 is a non-lethal skin condition that occurs after treatment 22 for VL in 5-20% of cases in the ISC, and less frequently in 23 individuals who report no history of prior VL (8, 10). It is 24 characterised by skin lesions of di ering severity and parasite 25 load, ranging from macules and papules (least severe, lowest 26 load) to nodules (most severe, highest load) (11). We estimate 27 the relative contributions of di erent disease states (VL, PKDL 28 and asymptomatic infection) to transmission, and quantify the 29 rate of spread of infection around infected individuals in space 30 and time by reconstructing transmission trees. Our analysis 31 provides insight into the spatiotemporal spread of visceral 32 leishmaniasis as well as quantitative estimates that can guide 33 the targeting of interventions, such as active case detection 34 and indoor residual spraying (IRS) of insecticide, around VL 35Although methods for analysing individual-level geo-located disease data have existed for some time, they have rarely been used to analyse endemic human diseases. Here we apply such methods to nearly a decade's worth of uniquely detailed epidemiological data on incidence of the deadly vector-borne disease visceral leishmaniasis (VL) and its secondary condition, post-kala-azar dermal leishmaniasis (PKDL), to quantify the spread of infection around cases in space and time by inferring who infected whom, and estimate the relative contribution of different infection states to transmission. Our findings highlight the key role long diagnosis delays and PKDL play in maintaining VL transmission. This detailed characterisation of the spatiotemporal transmission of VL will help inform targeting of interventions around VL and PKDL cases. : medRxiv preprint 131 sion (over and above that from being at zero distance from 132 an infectious individual) and with di erent pre-symptomatic 133 and asymptomatic relative infectiousness. Models with addi-134 tional within-household transmission fitted the data signifi-135 cantly better than those without according to the Deviance 136 Information Criterion (DIC) (32), and for these models fit 137 improved with increasing pre-symptomatic and asymptomatic 138 infectiousness, such that the best-fitting model had additional 139 within-household transmission and assumed 2% relative infec-140 tiousness of pre-symptomatic and asymptomatic individuals. 141 Here we present the results from this model (see SI Text for 142 results from the other models). 143 Parameter Estimates. We estimated the transmission model 144 parameters and unobserved data using the Markov Chain 145 Monte Carlo (MCMC) algorithm described in Materials and 146 Methods and SI Text. The posterior distributions obtained 147 for the model parameters are shown in Fig. S6 and the corre-148 sponding posterior modes and 95% credible intervals (CI) are\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Human astroviruses are small nonenveloped viruses with positive-sense single-stranded RNA genomes that contain three main open reading frames: ORF1a, ORF1b and ORF2. Astroviruses cause acute gastroenteritis in children worldwide and have been associated with encephalitis and meningitis in immunocompromised individuals. Through comparative genomic analysis of >400 astrovirus sequences, we identified a conserved \"ORFX\" overlapping the capsid-encoding ORF2 in genogroup I, III and IV astroviruses. ORFX appears to be subject to purifying selection, consistent with it encoding a functional protein product, termed XP. Using ribosome profiling of cells infected with human astrovirus 1, we confirm initiation at the ORFX AUG.XP-knockout astroviruses are strongly attenuated and after passaging can partly restore viral titer via pseudo-reversions, thus demonstrating that XP plays an important role in virus growth.To further investigate XP, we developed an astrovirus replicon system. We demonstrate that XP has only minor effects on RNA replication and structural protein production. Instead, XP associates with the plasma membrane with an extracellular N-terminus topology and promotes efficient virus release. Using two different assays, we show that expression of human or related astrovirus XPs leads to cell permeabilization, suggesting a viroporin-like activity. The discovery of XP advances our knowledge of these important human viruses and opens a new direction of research into astrovirus replication and pathogenesis.They are considered to be the second most common cause of gastroenteritis in infants after rotavirus 1 . Recently, and especially in immunocompromised individuals, astroviruses have also . CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/661579 doi: bioRxiv preprint been associated with extra-intestinal infections including fatal meningitis and encephalitis 2 .Despite their undoubted importance, astroviruses still represent one of the least studied groups of human positive-sense RNA viruses. The genome contains three main open reading frames:ORF1a encoding a nonstructural polyprotein (nsP1a), ORF1b encoding the RNA-dependent RNA polymerase (RdRp), and ORF2 encoding the capsid protein (CP) (Fig. 1A) 3 . ORF1a and ORF1b are translated from the genomic RNA (gRNA), with expression of ORF1b depending on programmed ribosomal frameshifting whereby a proportion of ribosomes translating ORF1a make a −1 nucleotide shift into ORF1b. Frameshifting occurs at a conserved A_AAA_AAC sequence within the ORF1a/ORF1b overlap region and depends on a 3′-adjacent stimulatory RNA stem-loop structure 4 . ORF2 is translated from a subgenomic RNA (sgRNA) that is produced during virus infection.Previously, using comparative genomics we identified a conserved fourth ORF (ORFX) in genogroup I astroviruses, which appears to be subject to purifying selection and therefore is likely to encode a functional protein product, termed the X protein (XP, 12 kDa, 112 aa) 5 .ORFX overlaps the 5′ region of the capsid-encoding ORF2 and is predicted to be translatable via ribosomal leaky scanning, which is expected to be enhanced by the very short leader on the sgRNA 3,6,7 . However, despite the comparative genomic predictions, the existence, relevance, and function of XP have never been demonstrated. Here we perform a functional molecular dissection of ORFX, providing insight into the simultaneous orchestration of several overlapping functional elements in a small human-pathogenic RNA virus. We show that ORFX is translated during virus infection, XP knockout mutant viruses are highly attenuated, and XP is a transmembrane (TM) protein that promotes efficient virus release via a viroporin-like activity. Through an extended bioinformatic analysis, we predict that related XP proteins are widely encoded in other mammalian astroviruses including the divergent \"MLB\" group of human-infecting astroviruses.Comparative genomic analysis reveals a potential ORFX in genogroup I, III and IV astroviruses.Since there are many more astrovirus sequences available now than in 2010, we began by repeating our previous comparative genomic analysis but this time also extending to other astrovirus genogroups. All mammalian astrovirus complete or nearly complete genome sequences were obtained from the\n",
      "\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "11\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Influenza is a major global public health threat as a result of its highly pathogenic variants, large 27 zoonotic reservoir, and pandemic potential. Metagenomic viral sequencing offers the potential of 28 a diagnostic test for influenza which also provides insights on transmission, evolution and drug 29 resistance, and simultaneously detects other viruses. We therefore set out to apply Oxford 30 Nanopore Technology to metagenomic sequencing of respiratory samples. We generated 31 influenza reads down to a limit of detection of 10 2 -10 3 genome copies/ml in pooled samples, 32 observing a strong relationship between the viral titre and the proportion of influenza reads (p = 33 4.7x10 -5 ). Applying our methods to clinical throat swabs, we generated influenza reads for 27/27 34 samples with high-to-mid viral titres (Cycle threshold (Ct) values <30) and 6/13 samples with low 35 viral titres (Ct values 30-40). No false positive reads were generated from 10 influenza-negative 36 samples. Thus Nanopore sequencing operated with 83% sensitivity (95% CI 67-93%) and 100% 37 specificity (95% CI 69-100%) compared to the current diagnostic standard. Coverage of full 38 length virus was dependent on sample composition, being negatively influenced by increased 39 host and bacterial reads. However, at high influenza titres, we were able to reconstruct >99% 40 complete sequence for all eight gene segments. We also detected Human Coronavirus and 41 generated a near complete Human Metapneumovirus genome from clinical samples. While 42 further optimisation is required to improve sensitivity, this approach shows promise for the 43 Nanopore platform to be used in the diagnosis and genetic analysis of influenza and other 44 respiratory viruses. 45 46\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Strict interventions were successful to control the novel coronavirus (COVID-19) outbreak in China. As transmission intensifies in other countries, the interplay between age, contact patterns, social distancing, susceptibility to infection and disease, and COVID-19 dynamics remains unclear. To answer these questions, we analyze contact surveys data for Wuhan and Shanghai before and during the outbreak and contact tracing information from Hunan Province. Daily contacts were reduced 7-9 fold during the COVID-19 social distancing period, with most interactions restricted to the household. Children 0-14 years were 59% (95% CI 7-82%) less susceptible than individuals 65 years and over. A transmission model calibrated against these data indicates that social distancing alone, as implemented in China during the outbreak, is sufficient to control COVID-19. While proactive school closures cannot interrupt transmission on their own, they reduce peak incidence by half and delay the epidemic. These findings can help guide global intervention policies.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Rapid production and publication of pathogen genome sequences during emerging disease outbreaks provide crucial public health information. In resource-limited settings, especially near an outbreak epicenter, conventional deep sequencing or bioinformatics are often challenging. Here we successfully used metagenomic next generation sequencing on an iSeq100 Illumina platform paired with an open-source bioinformatics pipeline to quickly characterize Cambodia's first case of COVID-\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The fast and untraceable virus mutations take lives of thousands of people before the immune system can produce the inhibitory antibody. Recent outbreak of novel coronavirus infected and killed thousands of people in the world. Rapid methods in finding peptides or antibody sequences that can inhibit the viral epitopes of COVID-19 will save the life of thousands. In this paper, we devised a machine learning (ML) model to predict the possible inhibitory synthetic antibodies for Corona virus.We collected 1933 virus-antibody sequences and their clinical patient neutralization response and trained an ML model to predict the antibody response. Using graph featurization with variety of ML methods, we screened thousands of hypothetical antibody sequences and found 8 stable antibodies that potentially inhibit COVID-19. We combined bioinformatics, structural biology, and Molecular Dynamics (MD) simulations to verify the stability of the candidate antibodies that can inhibit the Corona virus.\n",
      "\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Estimating temporal changes in a target population from phylogenetic or count data is an important problem in ecology and epidemiology. Reliable estimates can provide key insights into the climatic and biological drivers influencing the diversity or structure of that population and evidence hypotheses concerning its future growth or decline. In infectious disease applications, the individuals infected across an epidemic form the target population. The renewal model estimates the effective reproduction number, R, of the epidemic from counts of its observed cases. The skyline model infers the effective population size, N, underlying a phylogeny of sequences sampled from that epidemic. Practically, R measures ongoing epidemic growth while N informs on historical caseload. While both models solve distinct problems, the reliability of their estimates depends on p-dimensional piecewise-constant functions. If p is misspecified, the model might underfit significant changes or overfit noise and promote a spurious understanding of the epidemic, which might misguide intervention policies or misinform forecasts. Surprisingly, no transparent yet principled approach for optimising p exists. Usually, p is heuristically set, or obscurely controlled via complex algorithms. We present a computable and interpretable p-selection method based on the minimum description length (MDL) formalism of information theory. Unlike many standard model selection techniques, MDL accounts for the additional statistical complexity induced by how parameters interact. As a result, our method optimises p so that R and N estimates properly adapt to the available data. It also outperforms comparable Akaike and Bayesian information criteria on several classification problems. Our approach requires some knowledge of the parameter space and exposes the similarities between renewal and skyline models.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Faced with the current large-scale public health emergency, collecting, sorting, and analyzing biomedical information related to the \"coronavirus\" should be done as quickly as possible to gain a global perspective, which is a basic requirement for strengthening epidemic control capacity.However, for human researchers studying the viruses and the hosts, the vast amount of information available cannot be processed effectively and in a timely manner, particularly when the scientific understanding may be limited, which can further lower the information processing efficiency. We present TWIRLS, a method that can automatically acquire, organize, and classify information. Additionally, independent functional data sources can be added to build an inference system using a machine-based approach, which can provide relevant knowledge to help human researchers quickly establish subject cognition and to make more effective decisions. TWIRLS can automatically analyze more than three million words in more than 14,000 literature articles in only 4 hours. Combining with generalized gene interaction databases creates a data interface that can help researchers to further analyze the information. Using the TWIRLS system, we found that an important regulatory factor angiotensin-converting enzyme 2 (ACE2) may be involved in the All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity. : medRxiv preprint host pathological changes on binding to the coronavirus after infection. After triggering functional changes in ACE2/AT2R, an imbalance in the steady-state cytokine regulatory axis involving the Renin-Angiotensin System and IP-10 leads to a cytokine storm.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "medRxiv preprint 3 generally longer incubation and serial interval of less severe cases suggests a high risk of longterm epidemic in the absence of appropriate control measures.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Governments around the world must rapidly mobilize and make difficult policy decisions to mitigate the COVID-19 pandemic. Because deaths have been concentrated at older ages, we highlight the important role of demography, particularly how the age structure of a population may help explain differences in fatality rates across countries and how transmission unfolds. We examine the role of age structure in deaths thus far in Italy and South Korea and illustrate how the pandemic could unfold in populations with similar population sizes but different age structures, showing a dramatically higher burden of mortality in countries with older versus younger populations. This powerful interaction of demography and current age-specific mortality for COVID-19 suggests that social distancing and other policies to slow transmission should consider both the age composition of local and national contexts as well as the social connectedness of older and younger generations. We also call for countries to provide case and fatality data disaggregated by age and sex to improve real-time targeted nowcasting.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Compound heterozygotes occur when different mutations at the same locus on both maternal and paternal chromosomes produce a recessive trait. Here we present the tool VarCount for the quantification of mutations at the individual level. We used VarCount to characterize compound heterozygous coding variants in patients with epileptic encephalopathy and in the 1000 genomes participants. The Epi4k data contains variants identified by whole exome sequencing in patients with either Lennox-Gastaut Syndrome (LGS) or Infantile Spasms (IS), as well as their parents. We queried the Epi4k dataset (264 trios) and the phased 1000 genomes data (2504 participants) for recessive variants. To assess enrichment, transcript counts were compared between the Epi4k and 1000 genomes participants using minor allele frequency (MAF) cutoffs of 0.5% and 1.0%, and including all ancestries or only probands of European ancestry. In the Epi4k participants, we found enrichment for rare, compound heterozygous mutations in six genes, including three involved in neuronal growth and development -PRTG (p=0.00086, 1% MAF, combined ancestries), TNC (p=0.0221% MAF, combined ancestries), and MACF1 (p=0.0245, 0.5% MAF, EU ancestry). Due the total number of transcripts considered in these analyses, the enrichment detected was not significant after correction for multiple testing and higher powered or prospective studies are necessary to validate the candidacy of these genes.However, PRTG, TNC, and MACF1 are potential novel recessive epilepsy genes and our results highlight that compound heterozygous mutations should be considered in sporadic epilepsy.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Background With the spread of COVID-19 from Wuhan, Hubei Province to other areas of the country, medical staff in Fever Clinics faced the challenge of identifying suspected cases among febrile patients with acute respiratory infections. We aimed to describe the prevalence and clinical features of COVID-19 as compared to pneumonias of other etiologies in a Fever Clinic in Beijing.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "With the emergence of 4 rd generation transmission, the prevention and treatment of the novel coronavirus disease 2019 has entered a new period.We aimed to report several changes in the clinical characteristics at admission of patients with COVID-19.Clinical records and laboratory results of patients suffering from COVID-19were retrospectively reviewed and matched with the admission dates to analyze the changes in characteristics at the onset of illness.. Of the 89 affected patients, 31 [34.8%] patients were admitted from January 16 to 22, and 58 [65.2%] were admitted from January 23 to 29. Patients were admitted with more systemic symptoms, such as fever (21 [67.7%] of 31), fatigue (13 [41.9%] of 31), and myalgia (7 [22.6%] of 31), before January 23. More patients (10 [32.3%] of 31) admitted before January 23 had a small amount of sputum production compared with a smaller proportion (4 [6.9%] of 58) of the patients admitted after January 23.Other symptoms, such as cough, nausea, diarrhea, and chest tightness, were not significantly different between the two groups. In addition, the group admitted before January 23 had a larger proportion of patients with reduced lymphocyte (13 [54.2%] of 24), CD3 (11 [54.4%] of 21), and CD8 (9 [42.9%] of 21) counts and elevated serum amyloid A (SAA, 18 [75%] of 24).Conclusions. The initial symptoms of recently infected patients seem more insidious, indicating that the new coronavirus may gradually evolve into a virus similar to influenza and latent in asymptomatic carriers for a long time.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The ability to detect an infectious agent in a widespread epidemic is crucial to the success of quarantine efforts in addition to sensitive and accurate screening of potential cases of infection from patients in a clinical setting. Enabling testing outside of sophisticated laboratories broadens the scope of control and surveillance efforts, but also requires robust and simple methods that can be used without expensive instrumentation. Here we report a method to identify SARS-CoV-2 (COVID-19) virus RNA from purified RNA or cell lysis using loop-mediated isothermal amplification (LAMP) using a visual, colorimetric detection. This test was additionally verified using RNA samples purified from respiratory swabs collected from COVID-19 patients in Wuhan, China with equivalent performance to a commercial RT-qPCR test while requiring only heating and visual inspection. This simple and sensitive method provides an opportunity to facilitate virus detection in the field without a requirement for complex diagnostic infrastructure.. CC-BY 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.is the (which was not peer-reviewed) The copyright holder for this preprint .\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Background Since Dec 2019, SARS-CoV-2 has caused about fifty thousand patients and over two thousand deaths in Wuhan, China. We reported characteristics of patients with COVID-19 during epidemic ongoing outbreak in Wuhan.Methods Data of COVID-19 patients with clinical outcome in a designated hospital in Wuhan, were retrospectively collected from electronic medical records. Characteristics were compared between patients who died or recovered, and between patients with different disease severity.Results By Feb 25, 2020, 403 patients were enrolled, 100 died and 303 recovered. Most of non-survivors tended to be males, old aged, or with chronic diseases. Duration from illness onset to admission was 9 (7-12) days. Patients with severe or critical illness had more days from onset to admission compared to those with ordinary illness. Lymphopenia, anemia, hypoproteinemia, and abnormal serum sodium were presented in 52.6%, 54.6%, 69.8%, and 21.8% cases, respectively.Patients who died or with severe/critical illness showed increased white blood cell and neutrophil count, serum total bilirubin, creatinine, hypersensitive troponin I, D-dimer, procalcitonin, and C-reactive protein, and decreased red blood cell, lymphocyte, platelet count, and serum albumin on admission compared to those who recovered or with ordinary illness. Complications of acute organ injury and secondary infection were common in patients with COVID-19, especially in non-survivors.Conclusions Multiple homeostasis disturbances were common in patients with severe or critical illness at admission. Early support should be provided, especially for old men with chronic disease, which is vital to control disease progression and reduce mortality of COVID-19 during epidemic ongoing outbreak.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Introduction: Outbreaks of infectious diseases trigger an increase in scientific research and output. Early in outbreaks, evidence is scarce, but it accumulates rapidly. We are continuously facing new disease outbreaks, including the new coronavirus (SARS-nCoV-2) in December 2019.The objective of this study was to describe the accumulation of evidence during the 2013-2016 Zika virus (ZIKV) outbreak in the Pacific and the Americas related to aetiological causal questions about congenital abnormalities and Guillain-Barré syndrome.Methods: We hypothesised that the temporal sequence would follow a pre-specified order, according to study design. We assessed 1) how long it takes before findings from a specific study design appear, 2) how publication of preprints could reduce the time to publication and 3) how time to publication evolves over time.Results: We included 346 publications published between March 6, 2014 and January 1, 2019. In the 2013--2016 ZIKV outbreak, case reports, case series and basic research studies were published first. Case-control and cohort studies appeared between 400-700 days after ZIKV was first detected in the region of the study origin. Delay due to the publication process were lowest at the beginning of the outbreak. Only 4.6% of the publications was available as preprints.Discussion: The accumulation of evidence over time in new causal problems generally followed a hierarchy. Preprints reduced the delay to initial publication. Our methods can be applied to new emerging infectious diseases.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "For decades, cationic polymer nanoparticles have been investigated for nucleic acid delivery. Despite promising in vitro transfection results, most formulations have failed to translate into the clinic due to significant in vivo toxicity -especially when delivered intravenously. To address this significant problem, we investigated the detailed mechanisms that govern the complex in vivo systemic toxicity response to common polymeric nanoparticles.We determined that the toxicity response is material dependent. For branched polyethylenimine (bPEI) nanoparticles -toxicity is a function of multiple pathophysiological responses -triggering of innate immune sensors, induction of hepatic toxicity, and significant alteration of hematological properties. In contrast, for chitosan-based nanoparticles -systemic toxicity is primarily driven through innate immune activation. We further identified that modification of primary amines to secondary and tertiary amines using the small molecule imidazole-aceticacid (IAA) ameliorates in vivo toxicity from both nanocarriers by different, material-specific mechanisms related to Toll-like receptor 4 activation (for bPEI) and complement activation driven neutrophil infiltration (for chitosan), respectively. Our results provide a detailed roadmap for evaluating in vivo toxicity of nanocarriers and identifies potential opportunities to reduce toxicity for eventual clinical translation.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "Our society is currently experiencing an unprecedented challenge, managing and containing an outbreak of a new coronavirus disease known as COVID-19. While China -where the outbreak started -seems to have been able to contain the growth of the epidemic, different outbreaks are nowadays being detected in multiple countries. Much is currently unknown about the natural history of the disease, such as a possible asymptomatic spreading or the role of age in both the susceptibility and mortality of the disease. Nonetheless, authorities have to take action and implement contention measures, even if not everything is known. To facilitate this task, we have studied the effect of different containment strategies that can be put into effect. Our work specifically refers to the situation in Spain as of February 28th, 2020, where a few dozens of cases have been detected. We implemented an SEIR-metapopulation model that allows tracing explicitly the spatial spread of the disease through data-driven stochastic simulations. Our results are in line with the most recent recommendations from the World Health Organization, namely, that the best strategy is the early detection and isolation of individuals with symptoms, followed by interventions and public recommendations aimed at reducing the transmissibility of the disease, which although not efficacious for disease eradication, would produce as a second order effect a delay of several days in the raise of the number of infected cases.. CC-BY-NC-ND 4.0 International license It is made available under a is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.(which was not peer-reviewed) The copyright holder for this preprint .\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "17 years after the SARS-CoV epidemic, the world is facing the COVID-19 pandemic.COVID-19 is caused by a coronavirus named SARS-CoV-2. Given the most optimistic projections estimating that it will take more than a year to develop a vaccine, our best short term strategy may lie in identifying virus-specific targets for small molecule interventions. All coronaviruses utilize a molecular mechanism called -1 PRF to control the relative expression of their proteins. Prior analyses of SARS-CoV revealed that it utilizes a structurally unique three-stemmed mRNA pseudoknot to stimulate high rates of -1 PRF, that it also harbors a -1 PRF attenuation element. Altering -1 PRF activity negatively impacts virus replication, suggesting that this molecular mechanism may be therapeutically targeted. Here we present a comparative analysis of the original SARS-CoV and SARS-CoV-2 frameshift signals. Structural analyses reveal that the core -1 PRF signal, composed of the U UUA AAC slippery site and three-stemmed mRNA pseudoknot is highly conserved. In contrast, the upstream attenuator hairpin is less well conserved. Functional assays revealed that both elements promote similar rates of -1 PRF and that silent coding mutations in the slippery site strongly ablate -1 PRF activity. We suggest that molecules that were previously identified as inhibiting SARS-CoV mediated -1 PRF may serve as lead compounds to counter the current pandemic.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "ATP-dependent chromatin remodelling proteins represent a diverse family of proteins that share ATPase domains that are adapted to regulate protein-DNA interactions. Here we present structures of the yeast Chd1 protein engaged with nucleosomes in the presence of the transition state mimic ADP-beryllium fluoride. The path of DNA strands through the ATPase domains indicates the presence of contacts conserved with single strand translocases and additional contacts with both strands that are unique to Snf2 related proteins. The structure provides connectivity between rearrangement of ATPase lobes to a closed, nucleotide bound state and the sensing of linker DNA. Two turns of linker DNA are prised off the surface of the histone octamer as a result of Chd1 binding, and both the histone H3 tail and ubiquitin conjugated to lysine 120 are re-orientated towards the unravelled DNA. This indicates how changes to nucleosome structure can alter the way in which histone epitopes are presented.\n",
      "\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "While the ecology of infectious disease is a rich field with decades worth of empirical evidence and theory, there are aspects that remain relatively under-examined. One example is the importance of the free-living survival stage of certain pathogens, and especially is cases where they are transmitted indirectly between hosts through an environmental reservoir intermediate. In this study, we develop an integrated, broadly applicable mathematical method to examine diseases fitting this description-the waterborne, abiotic and other indirectly transmitted (W.A.I.T.) infection framework. To demonstrate its utility, we construct realistic models of two very different epidemic scenarios: cholera in a densely populated setting with limited access to clean drinking water and hepatitis C virus in an urban setting of injection-drug users. Using these two exemplars, we find that the W.A.I.T. model fortifies the centrality of reservoir dynamics in the \"sit and wait\" infection strategy, and provides a way to simulate a diverse set of intervention strategies.\n",
      "\n",
      "\n",
      "\u001b[1mAbstract\u001b[0m\n",
      "The cutting-edge technology vaccinomics is the combination of two topics immunogenetics and immunogenomics with the knowledge of systems biology and immune profiling for designing vaccine against infectious disease. In our present study, an epitope-based peptide vaccine against nonstructural protein 4 of beta coronavirus, using a combination of B cell and T cell epitope predictions, followed by molecular docking methods are performed. Here, protein sequences of homologous nonstructural protein 4 of beta coronavirus are collected and conserved regions present in them are investigated via phylogenetic study to determine the most immunogenic part of protein. From the identified region of the target protein, the peptide sequence IRNTTNPSAR from the region ranging from 38-47 and the sequence PTDTYTSVYLGKFRG from the positions of 76-90 are considered as the most potential B cell and T cell epitopes respectively. Furthermore, this predicted T cell epitopes PTDTYTSVY and PTDTYTSVYLGKFRG interacted with MHC allelic proteins HLA-A*01:01 and HLA-DRB5*01:01 respectively with the low IC 50 values. These epitopes are perfectly fitted into the epitope binding grooves of alpha helix of MHC I molecule and MHC II molecule with binding energy scores -725.0 Kcal/mole and -786.0 Kcal/mole respectively, showing stability in MHC molecules binding. This MHC restricted epitope PTDTYTSVY also showed a good conservancy of 50.16% in world population coverage. This MHC I HLA-A*01:01 allele is present among 58.87% of Chinese population also. Therefore, the epitopes IRNTTNPSAR and PTDTYTSVYLGKFRG may be considered as potential peptides for peptide-based vaccine for coronavirus after further experimental study.\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#By method titles(), abstracts() and bodies() you can access to certain elements of the paper\n",
    "paper_abstracts = parser.abstracts()\n",
    "for abstract in paper_abstracts[Dataset.BIORXIV].values():\n",
    "    print(abstract)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
